<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        
        <title>Charleston Chiang Research Group</title>
<!--
The Story Theme
http://www.templatemo.com/tm-480-story
-->
        <!-- load stylesheets -->
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400">  <!-- Google web font "Open Sans" -->
        <link rel="stylesheet" href="css/bootstrap.min.css">                                                <!-- Bootstrap style -->
        <link rel="stylesheet" href="css/flexslider.css">                                                   <!-- Flexslider style -->       
        <link rel="stylesheet" href="css/templatemo-style.css">                                             <!-- Templatemo style -->

        <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
        <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
        <!--[if lt IE 9]>
          <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
          <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->

<!-- added on 11/9/2024 for ADA accessibility -->
<style>
a {
  color: #0265BB;
<!--  text-decoration: underline; -->
}
</style>
<!-- end of added styles -->

    </head>
    
    <body> <!-- originally  :body class="page":, now inserted another :div: in between for banner. Need to remove the last :/div: before :footer: -->
<div class="banner">
<a href="http://www.usc.edu"><img src="img/usc-shield-name-white.png" height=38px align="right" alt="Link to USC"/></a>
</div>
<div class="page">
            
            <!-- background images -->
            <div class="page-bg-imgs-list">
	      <img src="img/Wordcloud_20210918.jpg" id="page-1-img" class="main-img" alt="About Us">
	      <img src="img/MutantTakeover20180515.jpg" id="page-2-img" alt="Research">
	      <img src="img/ZhengheMap20250808.jpg" id="page-3-img" alt="Team">
	      <img src="img/Students_11-13-19_015.jpg" id="page-4-img" alt="Teaching">
	      <img src="img/USC6.jpg" id="page-5-img" alt="Contact">
	      <img src="img/Coin20171208.jpg" id="page-6-img" class="main-img" alt="Join Us">
            </div>
            
            <div class="container-fluid">

                <div class="row">
                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-5 col-xl-5">
                        
                        <div class="header">

                            <!-- site title -->
                            <header class="box box-white">
                                <a href="javascript:void(0)" class="js-site-title">
                                    <h1 class="box-text site-title-text">Chiang.Lab</h1>    
                                </a>            
                            </header>
                            
                            <!-- site navigation -->
                            <nav class="js-nav">
                                <ul class="nav-items-container">

                                    <li data-nav-item-id="page-1" class="block-keep-ratio block-keep-ratio-1-1 block-width-half box box-white box-nav-item js-nav-item pull-xs-left">
                                        <a href="#page-1" class="block-keep-ratio-content box-nav-item-link">
                                            <span class="box-text box-text-nav-item flexbox-center">
				            About.Us
				            </span>
                                        </a>
                                    </li>

                                    <li data-nav-item-id="page-2" class="block-keep-ratio block-keep-ratio-1-1 block-width-half box box-white box-nav-item js-nav-item pull-xs-right">
                                        <a href="#page-2" class="block-keep-ratio-content box-nav-item-link">
                                            <span class="box-text box-text-nav-item flexbox-center">
				            Research
				            </span>
                                        </a>
                                    </li>

                                    <li data-nav-item-id="page-3" class="block-keep-ratio block-keep-ratio-1-1 block-width-half box box-white box-nav-item js-nav-item pull-xs-left">
                                        <a href="#page-3" class="block-keep-ratio-content box-nav-item-link">
                                            <span class="box-text box-text-nav-item flexbox-center">
				            Team
				            </span>
                                        </a>
                                    </li>

				    <li data-nav-item-id="page-4" class="block-keep-ratio block-keep-ratio-1-1 block-width-half box box-white box-nav-item js-nav-item pull-xs-right">
                                        <a href="#page-4" class="block-keep-ratio-content box-nav-item-link">
                                            <span class="box-text box-text-nav-item flexbox-center">
				            Teaching
				            </span>
                                        </a>
                                    </li>

				    <li data-nav-item-id="page-5" class="block-keep-ratio block-keep-ratio-1-1 block-width-half box box-white box-nav-item js-nav-item pull-xs-left">
                                        <a href="#page-5" class="block-keep-ratio-content box-nav-item-link">
                                            <span class="box-text box-text-nav-item flexbox-center">
				            Contact
				            </span>
                                        </a>
				    </li>
				    
				    <li data-nav-item-id="page-6" class="block-keep-ratio block-keep-ratio-1-1 block-width-half box box-white box-nav-item js-nav-item pull-xs-right">
                                        <a href="#page-6" class="block-keep-ratio-content box-nav-item-link">
                                            <span class="box-text box-text-nav-item flexbox-center">
				            Join.Us
				            </span>
                                        </a>
				    </li>


<!--				    <li data-nav-item-id="page-7" class="block-keep-ratio block-keep-ratio-1-1 block-width-half box box-white box-nav-item js-nav-item pull-xs-left">
                                        <a href="#page-7" class="block-keep-ratio-content box-nav-item-link">
                                            <span class="box-text box-text-nav-item flexbox-center">
				            Activities
				            </span>
                                        </a>
                                    </li> -->
                                </ul>
                            </nav>    
                        </div> <!-- .header -->

                    </div>

                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-7 col-xl-7">
                        <div class="content-wrapper js-content-wrapper">
                            <!-- about -->
                            <section data-page-id="page-1" class="content js-content">
                                
                                <header class="box box-black margin-b-20">
                                    <h2 class="box-text page-title-text">About Us</h2>
                                </header>
                                
                                <div class="content-text">
				  <p>Welcome to our Research Group's webpage!</p>
				  <p>We are a group of human geneticists and computational biologists. We utilize cutting-edge analytic tools to address questions at the intersection of human medical and population genetics. These insights will be critical for future medical genetics studies and in practicing personalized medicine.</p>
				  <p>For new and current members of the group, please check out <a href="resources.html" target="_blank"><b>Lab Expectations and Resources</b></a> and the <a href="labactivities.html" target="_blank"><b>Lab Activities</b></a> for what we have been up to outside of doing rigorous research!</p>
				  <p>Short video when the lab turned <b><a href="https://x.com/CharlestonCWKC/status/1730531991232344225?s=20" target="_blank">6 years old</a></b> in Dec 2023!</p>

				  <b><h4>Recent News</h4></b>
				  <p><b><i>8.2025</i></b>: Our method for a genealogy-based approach to elucidate ancestry-specific population structure is published in <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002-9297(25)00247-2" target="_blank">American Journal of Human Genetics</a>! Led by <b>Ji Tang</b>, he was also <a href="https://www.ashg.org/ajhg/inside-ajhg-a-chat-with-ji-tang/" target="_blank">interviewed</a> for his experience working on this project. Congrats Ji!</p>
				  <p><b><i>7.2025</i></b>: New member joining the lab! <b>Subarna Biswas</b> was a trained and practicing surgeon at USC but now would like to focus more in research. She has joined the lab as a Research Scientist. Welcome Subarna!</p>
				  <p><b><i>4.2025</i></b>: Our method for demographic inference, <i>gLike</i>, is published in <a href="https://www.nature.com/articles/s41588-025-02129-x" target="_blank">Nature Genetics</a>! Led by <b>Caoqi Fan</b> with help from <b>Jordan Cahoon</b> and <b>Bryan Dinh</b>, <i>gLike</i> can accurately infer demographic parameters of complex admixed history using genealogical trees. The code can be found <a href="https://github.com/Ephraim-usc/glike" target="_blank">here</a>. Congrats Caoqi!</p> 
				  <p><b><i>4.2025</i></b>: Our study on evaluating the accuracy of polygenic score models for BMI and Type-2 Diabetes in Native Hawaiians is published on <a href="https://www.nature.com/articles/s42003-025-08050-7" target="_blank">Communications Biology</a>! Led by <b>Ying-Chu Lo</b> with big assists from <b>He Tian</b>, <b>Tsz Fung Chan</b>, <b>Soyoung Jeon</b>, <b>Kimberli Alatorre</b>, and <b>Bryan Dinh</b>, we presented the first systematic evaluation of polygenic score models in this population.</p>
				  <p><b><i>4.2025</i></b>: New member joining the lab! <b>Leqi Tian</b>, a current first year student in the PIBBS Molecular Medicine program will be joining us for her dissertation research. Welcome Leqi!</p>
				  <p><b><i>3.2025</i></b>: <b>Charleston Chiang</b> presented the interim result of the Saudi Arabian project at the American Association of Biological Anthropologists meeting in Baltimore, MD.</p>
				  <p><b><i>1.2025</i></b>: A triple play to start the new year!
				  <p>&nbsp &nbsp &nbsp &nbsp<b>Ji Tang</b> preprinted his method of <a href="https://www.biorxiv.org/content/10.1101/2025.01.10.632475v1" target="_blank">ancestry-specific eGRM to reveal population structure in admixed populations</a> (tweetorial <a href="https://x.com/JiTang1024/status/1879746714174988334" target="_blank">here</a>).
				  <p>&nbsp &nbsp &nbsp &nbsp<b>Jalen Langie</b> preprinted the first chapter of her dissertation investigating the <a href="https://www.medrxiv.org/content/10.1101/2025.01.14.25320563v1" target="_blank">impact of Indigenous American ancestry on risk of childhood leukemia among Latino children</a> (tweetorial <a href="https://x.com/j__langie/status/1884853107118817745" target="_blank">here</a>).
				  <p>&nbsp &nbsp &nbsp &nbsp<b>Dorcus Malomane</b> preprinted her study to understand <a href="https://www.biorxiv.org/content/10.1101/2025.01.10.632500v1" target="_blank">how demographic history and cultural practices shaped the pattern of variation in Saudi Arabians</a> (descriptions <a href="https://www.linkedin.com/posts/dr-dorcus-kholofelo-malomane-phd-5a37b914_patterns-of-population-structure-and-genetic-activity-7300338929335947265-BXop?utm_source=share&utm_medium=member_desktop&rcm=ACoAAAN8IpABOZG4GLnRtevyiK67wRU8DRRyD3A" target="_blank">here</a>).</p>
				  <p><b><i>11.2024</i></b>: Our lab is again well-represented at the annual American Society of Human Genetics meeting, including four posters and <a href="https://x.com/CharlestonCWKC/status/1854742729109995864" target="_blank">a talk</a> from <b>Ji Tang</b> on his method of using ancestry-specific eGRM to reveal ancestry-specific population structure! <b>Charleston Chiang</b> also moderated a <a href="https://www.linkedin.com/posts/olowo-abiodun-7aa11118_it-was-a-privilege-to-moderate-the-novel-ugcPost-7260284639653642241-o3kN?utm_source=share&utm_medium=member_desktop" target="_blank">session</a>. It was great to see old friends and meet new ones, and there was even a snow storm in Denver! Check out the photos and links to posters and presentations <a href="https://x.com/CharlestonCWKC/status/1854027747049246936" target="_blank">here</a>!</p>
				  <p><b><i>10.2024</i></b>: <b>Charleston Chiang</b> received the Keck School of Medicine Dean's Award for Early Career <a href="https://keck.usc.edu/news/keck-school-of-medicine-celebrates-faculty-achievements-at-annual-awards-ceremony/" target="_blank">Excellence in Mentoring</a>! Thank you to all of the present and former trainees that made this work so enjoyable!</p>
				  <p><b><i>10.2024</i></b>: <b>Jalen Langie</b> successfully defended her qualifying exam proposal and has now officially advanced to candidacy. Congrats Jalen!</p>
				  <p><b><i>10.2024</i></b>: New members joining the lab! <b>Kimberli Alatorre</b>, who was a recent graduate of UCSB and was an intern in the group, has stayed on as a Programmer Analyst for the upcoming year! <b>Eaaswar Muthukrishnan</b> joined as a Research Associate with extensive experience in both academia and industry conducting population genetics research. Welcome Kimberli and Eaaswar!</p>
				  <p><b><i>7.2024</i></b>: <b>Charleston Chiang</b> gave a talk on the genealogy-based framework to estimate population structure and demographic history at Society of Molecular Biology and Evolution (SMBE) meeting in Puerto Vallarta, Mexico.</p>
				  <p><b><i>5.2024</i></b>: <b>Jordan Cahoon</b> has <a href="https://x.com/CharlestonCWKC/status/1788689423023919284" target="_blank">graduated</a> with her Bachelor in Computer Science! She accomplished so much during her three years in our lab, including winning a Barry Goldwater Scholarship, The Discovery Scholar at USC, and the Google Alumni Award from QCB. Jordan will be starting her Ph.D. in Biomedical Data Science at Stanford in the Fall. Congrats Jordan!</p>
				  <p><b><i>5.2024</i></b>: The Imputation-around-the-world project is published in the May issue of <a href="https://www.cell.com/ajhg/abstract/S0002-9297(24)00084-3" target="_blank">American Journal of Human Genetics</a>! Led by <b>Jordan Cahoon</b>, with help from <b>Camellia Rui</b>, <b>Echo Tang</b>, <b>Chris Simons</b>, <b>Jalen Langie</b>, <b>Ying-Chu Lo</b>, and <b>Minhui Chen</b>. In this paper we documented the continuing disparity in imputation accuracy across the world. You can also explore the results <a href="https://www.jordancahoon.com/iaw/map.html" target="_blank">here</a>!</p>
				  

				  
				  <p><br />Read our older news <a href="oldnews.html" target="_blank">here</a>.</p>

                                </div>            

                            </section> <!-- #about -->

				
                            <!-- Research -->
                            <section data-page-id="page-2" class="content js-content">
                                
                                <header class="box box-black margin-b-20">
                                    <h2 class="box-text page-title-text">Research</h2>
                                </header>
                                
                                <div class="content-text">
                                    <p>The overarching theme of our research group is to <b>use genetic approaches to understand how evolutionary forces shaped the genetic architecture of complex traits within and between populations</b>. To this end, we have been involved in a number of past and ongoing medical genetics studies in mapping genetic loci underlying human complex traits. We are also continually interested in investigating the evolutionary forces, namely demography and selection, that shaped the pattern of genetic variability and phenotypic distribution. We are particularly interested in global human populations with unique histories and ancestries, and our successes result from collaborating with innovative colleagues and thriving in resourceful consortiums. Read a more detailed description of our work <a href="research.html" target="_blank">here</a>.</p>
                                </div>            

                            </section> <!-- #Research -->
				
				
                            <!-- Team -->
                            <section data-page-id="page-3" class="content content-gallery js-content">
                                
                                <header class="box box-black margin-b-20">
                                    <h2 class="box-text page-title-text">Team Members (in chronological order)</h2>
                                </header>
                                
                                <div class="content-text content-text-gallery">
                                    
                                    <!-- <p>Credits go to <a rel="nofollow" href="http://unsplash.com">Unsplash</a> for images.</p> -->

                                    <div class="flexslider-wrapper">

 
                                        <div id="carousel" class="flexslider">
                                            <ul class="slides">
                                                <li><img src="img/Chiang_01a.jpg" alt="Charleston Chiang" /></li>
						<li><img src="img/tszfung2.jpg" alt="Tsz Fung Chan" /></li>
						<li><img src="img/BryanDinhUpdated_2.jpg" alt="Bryan Dinh" /></li>
						<li><img src="img/Jalen2.jpg" alt="Jalen Langie" /></li>
						<li><img src="img/Xinran.jpg" alt="Xinran Wang" /></li>
						<li><img src="img/HeTian.jpg" alt="He Tian" /></li>
						<li><img src="img/JiTang.jpg" alt="Ji Tang" /></li>
						<li><img src="img/Indu2.jpg" alt="Indu Sharma" /></li>
						<li><img src="img/Eunice3.jpg" alt="Eunice Lee" /></li>
						<li><img src="img/Kimberli2.jpg" alt="Kimberli Alatorre" /></li>
						<li><img src="img/Eaaswar2.jpg" alt="Eaaswar Muthukrishnan" /></li>
						<li><img src="img/Leqi_IMG33_2.jpg" alt="Leqi Tian" /></li>
						<li><img src="img/SubarnaPic.jpg" alt="Subarna Biswas" /></li>
						<li><img src="img/noun_group_2064675_mod.jpg" alt="Lab" /></li>
						<li><img src="img/noun_graduation_652526_mod.jpg" alt="alumni" /></li>
						<li><img src="img/noun_55168_cc.jpg" alt="?" /></li>
                                            </ul>
                                        </div>  <!-- #carousel -->

					<div id="slider" class="flexslider2"> <!-- was originally "flexslider" only -->
					  <ul class="slides"><li>
					    <p><b>Charleston Chiang</b> is the principal investigator of the group. He is an Associate Professor (with tenure) in the Department of Population and Public Health Sciences at USC Keck School of Medicine and Department of Quantitative and Computational Biology at USC Dornsife College of Letters, Arts, and Sciences. He is also the Associate Director for the Center of Genetic Epidemiology. He is broadly interested in using genetic approaches to understand how natural selection and demographic history shaped the variations in complex traits within and between global human populations.</p>
					        
					    <p>Prior to setting up his lab in 2018, he was an NRSA postdoctoral fellow working with Nelson Freimer and John Novembre at UCLA. Charleston received his Ph.D. in Genetics from Harvard University where he worked with Joel Hirschhorn.</p>
					        
					    <p>Charleston's <a href="doc/cv_CChiang_2025_07.pdf">CV</a> and
					    <a href="https://scholar.google.com/citations?user=qYy0_YwAAAAJ&hl=en" target="_blank">Google Scholar</a> profile<br />
					    ORCID: <a href="https://orcid.org/0000-0002-0668-7865" target="_blank">0000-0002-0668-7865</a><br />
					    Email: charleston [dot] chiang [at] med [dot] usc [dot] edu<br />
					    Follow him on social media:
					    <a href="https://bsky.app/profile/charleston.bsky.social" target="_blank">&#64charleston.bsky.social</a> (blusky) or 
					    <a href="https://twitter.com/CharlestonCWKC" target="_blank">&#64CharlestonCWKC</a> (twitter/X)<br /><br />
					    </p>
					    </li>

				<!--	    <li><p><b>Minhui Chen</b> is a postdoctoral scholar in the group. His research utilizes genomic data to study the evolutionary and demographic processes that shaped the genetic architecture of complex traits in Finland. He is interested in population genetics and statistical methodologies. Before joining the group, Minhui received his Ph.D. from China Agricultural University and Aarhus University in Denmark, working on selective sweeps, population admixture and adaptive introgression in domesticated animals.</p>
					    </li>


					    <li><p><b>Meng (Lemon) Lin</b> is a postdoctoral scholar in the group. She is interested in characterizing the genetic architecture of human complex traits. She is particularly interested in understanding component of architecture due to the non-additive or rare variations, and the interactions of evolutionary histories of diverse population and pattern of variation. Before joining the group, she received her PhD from SUNY Stony Brook working with <a href="http://hennlab.ucdavis.edu/" target="_blank">Dr. Brenna Henn</a>, where her work focused on the genetic landscape and evolution of skin pigmentation and height of the indigenous and admixed southern African population. Motivated by her former training, she is extending her interest to investigate the demographic history and population structure of Native Hawaiians, and how these insights can be leveraged for trait mapping empirically. She is also investigating the dominance components behind various clinical and anthropometric traits in a large Finnish cohort.</p>
					    </li>


					    <li><p><b>Soyoung (Elizabeth) Jeon</b> is a Ph.D student in the group (co-advised by <a href="https://sites.usc.edu/childhoodcancer/" target="_blank">Dr. Joe Wiemels</a>). She is pursuing her Ph.D. in the Cancer Biology and Genomics track through PIBBS at Keck School of Medicine. Her dissertation investigates the genetic architecture of childhood leukemia in a multi-ethnic cohort, examines the correlation between ancestry and the risk of childhood Leukemia, and construct and evaluates polygenic risk stratification models for leukemia. Generally, she is interested in understanding how human population history contributes to variations in susceptibility to disease and response to therapy. Before joining the group, she received her Bachelor of Science in Molecular and Cell Biology, with Medical Biology and Physiology track from University of California, Berkeley in 2017.</p>

					    <li><p><b>Soyoung (Elizabeth) Jeon</b> is a postdoc in the group. She received her Ph.D. in the Cancer Biology and Genomics track through PIBBS at Keck School of Medicine in 2023. Her dissertation investigated the genetic architecture of childhood leukemia in a multi-ethnic cohort, examines the correlation between ancestry and the risk of childhood Leukemia, and construct and evaluates polygenic risk stratification models for leukemia. Generally, she is interested in understanding how human population history contributes to variations in susceptibility to disease and response to therapy. She is pursuing these avenues of research in the Multiethnic Cohort, particularly focusing on the Native Hawaiian cohort. Prior to her Ph.D., she received her Bachelor of Science in Molecular and Cell Biology, with Medical Biology and Physiology track from University of California, Berkeley in 2017.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li><b>Jeon S</b>, Lo YC, Morimoto LM, Metayer C, Ma X, Wiemels JL, de Smith AJ, Chiang CWK†. “Evaluating Genomic Polygenic Risk Scores for Childhood Acute Lymphoblastic Leukemia in Latinos.” medRxiv 2023. doi: 10.1101/2023.06.08.23291167. <a href="https://www.medrxiv.org/content/10.1101/2023.06.08.23291167v1" target="_blank">link</a><br/><br/></li>
					      <li>Yu F, Cato LD, Weng C, Liggett LA, <b>Jeon S</b>, Xu K, Chiang CWK, Wiemels JL, Weissman JS, de Smith AJ, Sankaran VG. “Variant to function mapping at single-cell resolution through network propagation.” <b>Nat Biotechnol</b>. 2022 Jun 6. doi: 10.1038/s41587-022-01341-y. <a href="https://pubmed.ncbi.nlm.nih.gov/35668323/">link</a><br/><br/></li> 
					      <li><b>Jeon S</b>, de Smith AJ, Li S, Chen M, Chan TF, Muskens IS, Morimoto LM, Dewan AT, Mancuso N, Metayer C, Ma X, Wiemels JL, Chiang CWK. “Genome-wide trans-ethnic meta-analysis identifies novel susceptibility loci for childhood acute lymphoblastic leukemia.” <b>Leukemia</b>. 2022 Mar;36(3):865-868. doi: 10.1038/s41375-021-01465-1. Epub 2021 Nov 8. <a href="https://pubmed.ncbi.nlm.nih.gov/34750507/">link</a><br/><br/></li>
					      <li>Kachuri L, <b>Jeon S</b>, DeWan AT, Metayer C, Ma X, Witte JS, Chiang CWK, Wiemels JL, de Smith AJ. “Genetic determinants of blood cell traits influence susceptibility to childhood acute lymphoblastic leukemia.” <b>Am J Hum Genet</b>. 2021 Oct 7;108(10):1823-1835. doi: 10.1016/j.ajhg.2021.08.004. <a href="https://pubmed.ncbi.nlm.nih.gov/34469753/">link</a><br/><br/></li>
					      <li>de Smith AJ, Walsh KM, Morimoto LM, Francis SS, Hansen HM, <b>Jeon S</b>, Gonseth S, Chen M, Sun H, Luna-Fineman S, Antillón F, Girón V, Kang AY, Smirnov I, Shao X, Whitehead TP, Barcellos LF, Jolly KW, Healy J, Laverdière C, Sinnett D, Taub JW, Birch JM, Thompson PD, Pombo-de-Oliveira MS, Spector LG, DeWan AT, Mueller BA, Chiang CWK, Metayer C, Ma X, Wiemels JL. “Heritable variation at the chromosome 21 gene ERG is associated with acute lymphoblastic leukemia risk in children with and without Down syndrome.” <b>Leukemia</b>. 2019 Nov;33(11):2746-2751. doi: 10.1038/s41375-019-0514-9. <a href="https://pubmed.ncbi.nlm.nih.gov/31296947/">link</a><br/><br/></li>
					    </ol>
					    </p>
					    </li>
					    
					    <li><p><b>Hanxiao Sun</b> is a Master student in the group. She is pursuing her M.S. degree in the Biostatistics program at USC Keck School of Medicine. Currently, she is conducting admixture mapping of metabolic traits in a Native Hawaiian cohort, and evaluating local ancestry deconvolution approaches in this recently admixed population. Before joining the group, she receievd her Bachelor of Science in Pharmaceutical Sciences from College of Pharmaceutical Sciences, Zhejiang Unviersity in 2017.</p>
					    </li>
					    
					    <li><p><b>Caoqi (Ephraim) Fan</b> is a Ph.D student in the group since 2019. He is pursuing his Ph.D. in the Computational Biology and Bioinformatics program in the Department of Biological Sciences. Working closely also with Dr. Nicholas Mancuso in the Center for Genetic Epidemiology, Caoqi's current research focus is on incorporating genealogies and ancestral recombination graphs to re-define the genetic relationship matrix and to infer demographic parameters for human populations. Before joining the group, he received his Bachelor of Science in Biology on Bioinformatics track from Peking University (Beijing, China) in 2018.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li>Link V, Schraiber JG, <b>Fan C</b>, Dinh B, Mancuso N, Chiang CWK, Edge MD. “Tree-based QTL mapping with expected local genetic relatedness matrices.” <b>bioRxiv</b> 2023. doi: 10.1101/2023.04.07.536093. <a href="https://www.biorxiv.org/content/10.1101/2023.04.07.536093v1" target="_blank">link</a><br/><br/></li>
					      <li><b>Fan C</b>, Mancuso N*, Chiang CWK*. “A genealogical estimate of genetic relationships.” <b>Am J Hum Genet</b>. 2022 Apr 6:S0002-9297(22)00112-4. doi: 10.1016/j.ajhg.2022.03.016. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(22)00112-4">link</a><br/><br/></li>
					      <li>Darst BF, Hughley R, Pfennig A, Hazra U, <b>Fan C</b>, et al. “A rare germline HOXB13 variant contributes to risk of prostate cancer in men of African ancestry.” <b>Eur Urol</b>. 2022 Jan 11:S0302-2838(21)02271-5. doi: 10.1016/j.eururo.2021.12.023. <a href="https://pubmed.ncbi.nlm.nih.gov/35031163/">link</a><br/><br/></li>
					      </ol>
					      </p>
					      </li> -->

					    <li><p><b>Tsz Fung Chan</b> is a Ph.D student in the group since 2019. He is pursuing his Ph.D. in Epidemiology in the Department of Preventive Medicine. Co-advised by both Dr. Chiang and <a href="http://mancusolab.com" target="_blank">Dr. Nicholas Mancuso</a> in the Center for Genetic Epidemiology, he is currently developing methods to estimate heritability of complex traits explained by local ancestry and to evaluate stratification bias in admixture mapping studies. Before joining the group, Tsz Fung studied the genetics of Epstein-Barr virus under the guidance of Alan Chiang and Wanling Yang at the University of Hong Kong. He received his Bachelor of Biomedical Sciences (with a minor in Computer Sciences) and M.Phil in Bioinformatics from University of Hong Kong in 2017 and 2019, respectively.</p>

					    <p><b>Recent publications</b>:<br/>
					      <ol>
						<li>Langie J, <b>Chan TF</b>, Yang W, Kang AY, Morimoto L, Stram DO, Mancuso N, Ma X, Metayer C, Lupo PJ, Rabin KR, Scheurer ME, Wiemels JL, Yang JJ, de Smith AJ*, Chiang CWK*. “The impact of Indigenous American-like ancestry on risk of acute lymphoblastic leukemia in Hispanic/Latino children.” <b>medRxiv</b> 2025. <a href="https://www.medrxiv.org/content/10.1101/2025.01.14.25320563v1" target="_blank">preprint</a>.<br/><br/></li>
						<li>Lo YC, Tian H, <b>Chan TF</b>, Jeon S, Alatorre K, Dinh BL, Maskarinec G, Taparra K, Nakatsuka N, Yu M, Chen CY, Lin YF, Wilkens LR, Le Marchand L, Haiman CA, Chiang CWK. “Transferability of polygenic scores for anthropometric traits and Type II Diabetes to the Native Hawaiian population.” <b>Commun Biol.</b> 2025 Apr 23;8(1):651. <a href="https://www.nature.com/articles/s42003-025-08050-7" target="_blank">link</a><br/><br/></li>
						<li>de Smith AJ*, Wahlster L*, Jeon S*, Kachuri L, Black S, Cato LD, Nakatsuka N, <b>Chan TF</b>, Langie J, Mazumder S, Yang W, Gazal S, Eng C, Hu D, Buchard EG, Ziv E, Metayer C, Mancuso N, Yang JJ, Ma X, Wiemels JL, Yu F*, Chiang CWK*, Sankaran VG*. “A noncoding regulatory variant in IKZF1 increases acute lymphoblastic leukemia risk in Hispanic/Latino children.” <b>Cell Genom.</b> 2024 Apr 10;4(4):100526. <a href="https://www.cell.com/cell-genomics/fulltext/S2666-979X(24)00068-5" target="_blank">link</a><br/><br/></li>
						<li><b>Chan TF</b>, Rui X, Conti DV, Fornage M, Graff M, Haessler J, Haiman C, Highland HM, Jung SY, Kenny E, Kooperberg C, Le Marchand L, North KE, Tao R, Wojcik G, Gignoux CR, PAGE Consortium, Chiang CWK*, Mancuso N*. “Estimating heritability explained by local ancestry and evaluating stratification bias in admixture mapping from summary statistics.” <b>Am J Hum Genet.</b> 2023 Nov 2;110(11):1853-1862. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(23)00325-7">link</a><br/><br/></li>
						<li>Li S, Chiang CWK, Myint SS, Arroyo K, <b>Chan TF</b>, Morimoto L, Metayer C, de Smith AJ, Walsh KM, Wiemels JL. “Localized variation in ancestral admixture identifies pilocytic astrocytoma risk loci among Latino children.” <b>PLoS Genet</b>. 2022 Sep 7;18(9):e1010388. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010388">link</a><br/><br/></li>
						<li>Jeon S, de Smith AJ, Li S, Chen M, <b>Chan TF</b>, Muskens IS, Morimoto LM, Dewan AT, Mancuso N, Metayer C, Ma X, Wiemels JL, Chiang CWK. “Genome-wide trans-ethnic meta-analysis identifies novel susceptibility loci for childhood acute lymphoblastic leukemia.” <b>Leukemia</b>. 2022 Mar;36(3):865-868. <a href="https://pubmed.ncbi.nlm.nih.gov/34750507/">link</a><br/><br/></li>
						<li>Sun H*, Lin M*, Russell EM, Minster RL, <b>Chan TF</b>, Dinh BL, Naseri T, Reupena MS, Lum-Jones A, OLaGA Study Group, Cheng I, Wilkens LR, Le Marchand L, Haiman CA, Chiang CWK. “The impact of global and local Polynesian genetic ancestry on complex traits in Native Hawaiians.” <b>PLoS Genet</b>. 2021 Feb 11. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009273">link</a><br/><br/></li>
					      </ol>
					      </p>
					      </li>

				<!--	    <li><p><b>Samuel Sommerer</b> is an undergraduate student in the group since 2019. He is pursuing a B.S. degree in Computer Science at the USC Viterbi School of Engineering and a B.S. degree in Quantitative Biology at the USC Dornsife College of Letters, Arts and Sciences. He is interested in evolutionary population genetics and the intersection of computer science and biology. Currently, he is constructing a pipeline to study polygenic risk scores and allelic enrichment in diverse populations.</p>
					    </li>

					    <li><p><b>Sydney Rashid</b> is an undergraduate student in the group since 2020. She is pursuing a B.S. degree in Quantitative Biology at the USC Dornsife College of Letters, Arts and Sciences. She is studying to bridge the gap between computer science and genomics. Her research interests include investigating the underlying evolutionary forces driving genetic variation and disease susceptibility in understudied populations. Currently, she is working on calculating polygenic risk scores for children with Acute Lymphoblastic Leukemia using known loci from GWAS.</p>
					    </li> -->

					    <li><p><b>Bryan Dinh</b> is a Ph.D. student in the group since 2020. He is an F31 predoctoral fellow pursuing his Ph.D. in the Computational Biology and Bioinformatics program in the Department of Biological Sciences. Bryan is interested in algorithms and their applications to biology, genetics, and diverse populations. Supported by a F31 predoctoral fellowship from NHGRI, Bryan's dissertation focuses on constructing and evaluating genomic resources for Native Hawaiian community, with the goal to empower cutting edge genomic studies in this understudied population. Before joining the group, Bryan received a Master of Science in Computer Science and a Bachelor of Science in Mathematics and Economics from UCSD.</p>

					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li>Lo YC, Tian H, Chan TF, Jeon S, Alatorre K, <b>Dinh BL</b>, Maskarinec G, Taparra K, Nakatsuka N, Yu M, Chen CY, Lin YF, Wilkens LR, Le Marchand L, Haiman CA, Chiang CWK. “Transferability of polygenic scores for anthropometric traits and Type II Diabetes to the Native Hawaiian population.” <b>Commun Biol.</b> 2025 Apr 23;8(1):651. <a href="https://www.nature.com/articles/s42003-025-08050-7" target="_blank">link</a><br/><br/></li>
					      <li>Fan C, Cahoon JL, <b>Dinh BL</b>, Ortega-Del Vecchyo VD, Huber CD, Edge MD, Mancuso N, Chiang CWK. “A likelihood-based framework for demographic inference from genealogical trees.” <b>Nat Genet.</b> 2025 Apr;57(4):865-874.<a href="https://www.nature.com/articles/s41588-025-02129-x" target="_blank">link</a><br/><br/></li>
					      <li><b>Dinh BL</b>, Tang E, Taparra K, Nakatsuka N, Chen F, Chiang CWK. “Recombination map tailored to Native Hawaiians may improve robustness of genomic scans for positive selection.” <b>Hum Genet.</b> 2024 Jan;143(1):85-99. <a href="https://link.springer.com/article/10.1007/s00439-023-02625-2" target="_blank">link</a><br/><br/></li>
					      <li>Link V, Schraiber JG, Fan C, <b>Dinh BL</b>, Mancuso N, Chiang CWK, Edge MD. “Tree-based QTL mapping with expected local genetic relatedness matrices.” <b>Am J Hum Genet.</b> 2023 Dec 7;110(12):2077-2091. <a href="https://www.cell.com/ajhg/abstract/S0002-9297(23)00395-6" target="_blank">link</a><br/><br/></li>

					      <li>Sun H*, Lin M*, Russell EM, Minster RL, Chan TF, <b>Dinh BL</b>, Naseri T, Reupena MS, Lum-Jones A, OLaGA Study Group, Cheng I, Wilkens LR, Le Marchand L, Haiman CA, Chiang CWK. “The impact of global and local Polynesian genetic ancestry on complex traits in Native Hawaiians.” <b>PLoS Genet</b>. 2021 Feb 11. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009273">link</a><br/><br/></li>
					    </ol>
					    </p>
					  </li>
					    
				<!--	    <li><p><b>Camellia Rui</b> is an undergraduate student in the group since 2020. She is pursuing a B.A. degree in Mathematics at the USC Dornsife College of Letters, Arts, and Sciences, as well as a progressive M.S. degree in Biostatistics at Keck School of Medicine. Before joining the group, she has completed coursework in human biology, biochemistry, and mathematics. Previous to joining the lab, she had worked as an undergraduate research assistant and research intern in primate neurobiology. Currently, she is part of a team of researchers evaluating the current efficacy for imputing populations around the globe.</p>
					    <p>Personal Website: <a href="http://camelliarui.github.io/" target="blank_">http://camelliarui.github.io/</a></p>
					    </li>

					    <li><p><b>Echo Tang</b> is an undergraduate student in the group since 2021. She is pursuing a B.S. degree in Quantitative Biology from USC Dornsife College of Letters, Arts and Sciences and a minor in Health Policy. Before joining the group, she worked as a research assistant studying the genetics of aging and sex-specific effects of mutations. Currently, she is part of a team of researchers evaluating the current efficacy for imputing populations around the globe.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li>Dinh BL, <b>Tang E</b>, Taparra K, Nakatsuka N, Chen F, Chiang CWK. “Recombination map tailored to Native Hawaiians reduces false positive findings in genomic scans of positive selection.” <b>bioRxiv</b> 2023. doi: 10.1101/2023.07.12.548735v2. <a href="https://www.biorxiv.org/content/10.1101/2023.07.12.548735v2" target="_blank">link</a><br/><br/></li>
					      <li>Cahoon JL, Rui X*, <b>Tang E*</b>, Simons C, Langie J, Chen M, Lo YC, Chiang CWK. “Imputation Efficacy Across Global Human Populations.” <b>bioRxiv</b> 2023. doi: 10.1101/2023.05.22.541241. <a href="https://www.biorxiv.org/content/10.1101/2023.05.22.541241v1" target="_blank">link</a><br/><br/></li>
					    </ol>
					    </li>

					    <li><p><b>Jordan Cahoon</b> is an undergraduate student in the group since 2021. She is pursuing a B.S. degree in Computer Science at the USC Viterbi School of Engineering with a minor in Computational Biology and Bioinformatics. She is one of only ~400 undergraduates nationwide to be awarded the Barry Goldwater Scholarship. Jordan is interested in analyzing and visualizing genetic disease factors in diverse populations. Prior to joining the lab, she worked as an undergraduate research assistant in single-cell transcriptomics research. Supported by the Viterbi Merit Scholarship, she is leading a team of undergraduate researchers to evaluate the current efficacy for imputing populations around the globe, supporting algorithm development in the lab, and exploring the application of machine learning methods for population genetic inference using the ancestral recombination graph.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      	<li>Fan C, <b>Cahoon JL</b>, Dinh BL, Ortega-Del Vecchyo VD, Huber CD, Edge MD, Mancuso N, Chiang CWK. “A likelihood-based framework for demographic inference from genealogical trees.” <b>bioRxiv</b> 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.10.10.561787v1" target="_blank">link</a></br/><br/></li>
						<li><b>Cahoon JL</b>, Rui X*, Tang E*, Simons C, Langie J, Chen M, Lo YC, Chiang CWK. “Imputation Accuracy Across Global Human Populations.” <b>Am J Hum Genet.</b> 2024 May 2;111. <a href="https://www.cell.com/ajhg/abstract/S0002-9297(24)00084-3" target="_blank">link</a><br/><br/></li>
						<li>Sheng X, Xia L, <b>Cahoon JL</b>, Conti DV, Haiman CA, Kachuri L, Chiang CWK. “Inverted genomic regions between reference genome builds in humans impact imputation accuracy and decrease the power of association testing.” <b>HGG Adv.</b> 2022 Nov 11;4(1):100159. <a href="https://www.sciencedirect.com/science/article/pii/S2666247722000768?via%3Dihub">link</a><br/><br/></li>
					      </ol>
					      </p>
					    </li>

				    <li><p><b>Ying-Chu Lo</b> is a postdoctoral scholar in the group since 2021. She has had experience studying both fungal and plant models, exploring the genetic variation between populations. She obtained her Ph.D. in genetics and evolutionary biology in 2019 from the University of Paris-Saclay (University Paris XI) in Paris, France. In her Ph.D. she focused on using population genetics and comparative genomic methods to study fungi domestication and adaptation. After her PhD, she was a postdoc at ABRC Academia Sinica in Taiwan as a bioinformatic scientist, where she investigated the genes or gene clusters correlated to plant homeostasis under heavy metal stress. Currently, Ying-Chu is funded by a USC-Taiwan postdoctoral fellowship. She is focusing on understanding the genetic and non-genetic factors contributing to the poor transferability of polygenic risk score (PRS) risk prediction models between human populations.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li>Jeon S, <b>Lo YC</b>, Morimoto LM, Metayer C, Ma X, Wiemels JL, de Smith AJ, Chiang CWK†. “Evaluating Genomic Polygenic Risk Scores for Childhood Acute Lymphoblastic Leukemia in Latinos.” medRxiv 2023. doi: 10.1101/2023.06.08.23291167. <a href="https://www.medrxiv.org/content/10.1101/2023.06.08.23291167v1" target="_blank">link</a><br/><br/></li>
					      <li>Cahoon JL, Rui X*, Tang E*, Simons C, Langie J, Chen M, <b>Lo YC</b>, Chiang CWK. “Imputation Efficacy Across Global Human Populations.” <b>bioRxiv</b> 2023. doi: 10.1101/2023.05.22.541241. <a href="https://www.biorxiv.org/content/10.1101/2023.05.22.541241v1" target="_blank">link</a><br/><br/></li>
					    </ol>
					    </li>

					    <li><p><b>Christopher Simons</b> is an undergraduate student joining the group in 2021. He is pursuing a B.S. and M.S. in Quantitative Biology at USC. He enjoys learning about genetics, and hopes to improve upon current methods for data analysis using biological information. Prior to joining the lab, he participated in a student group which analyzed publicly available cancer datasets. He is currently leveraging sequencing data from diverse populations to directly assess the precision of imputation methods.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li>Cahoon JL, Rui X*, Tang E*, <b>Simons C</b>, Langie J, Chen M, Lo YC, Chiang CWK. “Imputation Efficacy Across Global Human Populations.” <b>bioRxiv</b> 2023. doi: 10.1101/2023.05.22.541241. <a href="https://www.biorxiv.org/content/10.1101/2023.05.22.541241v1" target="_blank">link</a><br/><br/></li>
					    </ol>
					    </li> -->

					    <li><p><b>Jalen Langie</b> is a Ph.D. student in the group since 2021. She is a F31 predoctoral fellow pursuing a Ph.D. in Epidemiology (Genetic Track) in the Department of Population and Public Health Sciences. Overall, she aims to investigate genetic contributions to disease through the lens of evolutionary history. Her interests include admixture mapping, fine mapping, and SNP-to-gene annotations in multiethnic populations. Prior to joining the lab, she received her Bachelor of Science in Microbiology from California State University Los Angeles in 2019. During undergraduate, she worked as a research assistant for three years studying fungal pathogenesis and regulatory pathways and completed an internship studying viral kinetics and chemokine expression at Johns Hopkins University.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li><b>Langie J</b>, Chan TF, Yang W, Kang AY, Morimoto L, Stram DO, Mancuso N, Ma X, Metayer C, Lupo PJ, Rabin KR, Scheurer ME, Wiemels JL, Yang JJ, de Smith AJ*, Chiang CWK*. “The impact of Indigenous American-like ancestry on risk of acute lymphoblastic leukemia in Hispanic/Latino children.” <b>medRxiv</b> 2025. <a href="https://www.medrxiv.org/content/10.1101/2025.01.14.25320563v1" target="_blank">preprint</a>.<br/><br/></li>
					      <li>Cahoon JL, Rui X*, Tang E*, Simons C, <b>Langie J</b>, Chen M, Lo YC, Chiang CWK. “Imputation Accuracy Across Global Human Populations.” <b>Am J Hum Genet.</b> 2024 May 2;111. <a href="https://www.cell.com/ajhg/abstract/S0002-9297(24)00084-3" target="_blank">link</a><br/><br/></li>
					      <li>de Smith AJ*, Wahlster L*, Jeon S*, Kachuri L, Black S, Cato LD, Nakatsuka N, Chan TF, <b>Langie J</b>, Mazumder S, Yang W, Gazal S, Eng C, Hu D, Buchard EG, Ziv E, Metayer C, Mancuso N, Yang JJ, Ma X, Wiemels JL, Yu F*, Chiang CWK*, Sankaran VG*. “A noncoding regulatory variant in IKZF1 increases acute lymphoblastic leukemia risk in Hispanic/Latino children.” <b>Cell Genom.</b> 2024 Apr 10;4(4):100526. <a href="https://www.cell.com/cell-genomics/fulltext/S2666-979X(24)00068-5" target="_blank">link</a>.<br/><br/></li>
					    </ol>
					    </li>

<!--					    <li><p><b>Dorcus Kholofelo Malomane</b> is a postdoctoral research scholar in the group since 2022. She is interested in exploring patterns of genetic variation across human populations and their relation to complex traits. Dorcus has a background in population and evolutionary genomics. She obtained her PhD in 2019 from the University of Goettingen, Germany. In her PhD work she studied how different geographic distributions, management and breeding practices have shaped the pattern of genetic variation in a wide variety of globally collected chicken populations. Furthermore, she investigated the genetic patterns of different genomic regions, pathways and genes in the chicken to explore their subjection to different evolutionary dynamics. Prior to joining the group, she was shortly a postdoctoral research associate at the University of Goettingen and her work was aimed at developing genetic markers for ash dieback disease. Her current project utilizes the genome-wide array data, whole exome and genome sequencing data to understand the impact of population structure and history on patterns of variation in Saudi Arabians.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li><b>Malomane DK</b>, Williams MP, Huber CD, Mangul S*, Abedalthagafi M*, Chiang CWK*. “Patterns of population structure and genetic variation within the Saudi Arabian population.” <b>bioRxiv</b> 2025. <a href="https://www.biorxiv.org/content/10.1101/2025.01.10.632500v1" target="_blank">preprint</a><br/><br/></li>
					    </ol>
					    </li> -->

					    <li><p><b>Xinran Wang</b> started as a Program Analyst in 2023 and is now a Ph.D. student in the lab since 2024. She is passionate about leveraging genomic data to gain insights into disease-trait associations, the evolution and architecture of complex traits, and the history of diverse human populations. At present, Xinran's work primarily focuses on evaluating imputation reference panel for Japanese and Native Hawaiian populations, as well as conducting GWAS and meta-analyses on hyperemesis gravidarum. Before joining the lab, she earned her Master of Science in Applied Biostatistics and Epidemiology from the University of Southern California and her Bachelor of Arts in Public Health from UC Irvine. During her master's program, Xinran gained valuable experience in GWAS, meta-analysis, and TWAS while working under the supervision of Dr. Nicholas Mancuso at USC and Dr. Marlena Fejzo at USC and UCLA.</p>
					    </li>

					    <li><p><b>He Tian</b> is a Ph.D. student in the group since 2023. She is pursuing her Ph.D. in Epidemiology in the Department of Population and Public Health Sciences. She is interested in Polygenic Risk Scores and cancers among ethnic minority populations. Before joining the lab, she received her Bachelor of Medicine in Preventive Medicine from Shandong University (Jinan, China) in 2020. During graduate school, she focused on studying the racial disparity in ovarian cancer and received her MPH in 2022.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li>Lo YC, <b>Tian H</b>, Chan TF, Jeon S, Alatorre K, Dinh BL, Maskarinec G, Taparra K, Nakatsuka N, Yu M, Chen CY, Lin YF, Wilkens LR, Le Marchand L, Haiman CA, Chiang CWK. “Transferability of polygenic scores for anthropometric traits and Type II Diabetes to the Native Hawaiian population.” <b>Commun Biol.</b> 2025 Apr 23;8(1):651. <a href="https://www.nature.com/articles/s42003-025-08050-7" target="_blank">link</a><br/><br/></li>
					    </ol>
					    </li>

					    <li><p><b>Ji Tang</b> obtained a Bioinformatics Ph.D. in July 2023 from the Southern Medical University of China. He was awarded as the Outstanding Graduate in his Ph.D., where he developed a deep learning method for identifying the genomic mutations favored by recent natural selection and investigated the association between adaptive evolution and disease susceptibility in human populations. In general, Ji is interested in investigating the influence of population history on the genetic architecture of traits/diseases in diverse human populations by developing computational methods (especially machine learning methods) and performing statistical analyses. Currently, he is focusing on developing a method to infer more accurate ARG (Ancestral Recombination Graph) when applied to empirical sequencing data with errors, and on demonstrating the enrichment of functionally important or consequential alleles in Polynesian-ancestry individuals, whose ancestors went through several bottlenecks and persisted in small population sizes for a long time.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li><b>Tang J</b>, Chiang CWK. “A genealogy-based approach for revealing ancestry-specific structures in admixed populations.” <b>Am J Hum Genet.</b> 2025 Aug 7;112(8):1906-1922. <a href="https://www.cell.com/ajhg/abstract/S0002-9297(25)00247-2" target="_blank">link</a><br/><br/></li>
					    </ol>
					    </li>

					    <li><p><b>Indu Sharma</b> is a postdoctoral research fellow since 2023. She is a highly accomplished researcher with a Ph.D. in Biotechnology from India, having previously worked as a Population Geneticist at New York University Abu Dhabi campus. Indu's research has predominantly centered on underrepresented and tribal populations, particularly those from Northwestern India and the Northwestern Himalayas. Her expertise in population genomics has allowed her to delve into the evolutionary impacts and gene-based associations of various complex disorders prevalent in South Asian endogamous groups. Her current research aims at unraveling the selection signatures within the Native Hawaiian population. Her work combines the precision of population genomics with a keen understanding of complex genetic traits.</p>
					    </li>

					    <li><p><b>Eunice Lee</b> is a senior research associate who joined the lab in 2023. She was born in Seoul, South Korea, and moved to and grew up in Southern California when she was young. She was trained as an environmental and genetic epidemiologist focusing on asthma in minority children and cardiovascular diseases in the adult U.S. population. Her main research interests are population genetics, computational toxicology, and gene and environment interactions using various statistical methods and machine learning algorithms. Outside of work, Eunice started rock climbing, taking pottery lessons, and enjoys outdoor activities such as hiking and camping.</p>
					    </li>

					    <li><p><b>Kimberli Alatorre</b> started as an intern in the lab and is now a programmer analyst since 2024. She is a recent graduate of the University of California, Santa Barbara, where she earned a Bachelor of Science in Biological Sciences. Aspiring to pursue a career in medicine, she aims to specialize in pediatrics, with a focus on serving underserved communities. Her academic interests include Polygenic Risk Scores (PRS) and the heritability of complex traits and diseases. A native of Los Angeles, Kimberli enjoys hiking, spending time at the beach, and running outdoors in her free time.</p>
					    <p><b>Recent publications</b>:<br/>
					    <ol>
					      <li>Lo YC, Tian H, Chan TF, Jeon S, <b>Alatorre K</b>, Dinh BL, Maskarinec G, Taparra K, Nakatsuka N, Yu M, Chen CY, Lin YF, Wilkens LR, Le Marchand L, Haiman CA, Chiang CWK. “Transferability of polygenic scores for anthropometric traits and Type II Diabetes to the Native Hawaiian population.” <b>Commun Biol.</b> 2025 Apr 23;8(1):651. <a href="https://www.nature.com/articles/s42003-025-08050-7" target="_blank">link</a><br/><br/></li>
					    </ol>
					    </li>

					    <li><p><b>Eaaswarkhanth Muthukrishnan</b> joined the lab in 2024 as a research associate. Originally from Pollachi, a charming town nestled near the Western Ghats of Tamil Nadu, south of India, Eaaswar brings a wealth of experience as a population geneticist with postdoc and staff scientist roles in diverse international labs. His research work includes uncovering complex evolutionary trajectories in shaping the human skin-barrier function, developing a robust analytical framework for analyzing genomic structural variants and identifying genetic bases of human phenotypic variation. Among his key research findings, Eaaswar has shed light on the evolutionary factors behind high rates of metabolic syndrome in Kuwait, characterized genetic variants related to lactase persistence in the Indian subcontinent, and tracked evolutionary trends in SARS-CoV-2 virus. His doctoral research established historical population movements by tracing sub-Saharan African genetic lineage within a non-tribal Indian population, a study that remains a key reference for understanding the origins and genetic diversity of South Asian Muslims. Currently, his research centers on the fine-scale population structure, genetic history and demographic patterns of Native Hawaiian and East Asian populations.</p>
					    </li>

					    <li><p><b>Leqi Tian</b> joined the lab as a Ph.D. student in 2025. She is pursuing a Ph.D. in the Programs in Biomedical and Biological Sciences (PIBBS) under the Molecular Medicine track. Her research focuses on complex trait evolution and signature of different modes of adaptation in different populations. Before joining the lab, she earned a Bachelor of Arts in Biological Sciences, Computer Science, and Statistical Science from Cornell University in 2024. As an undergraduate, she conducted research on gene drive population dynamics and Neanderthal introgression.</p>
					    </li>

					    <li><p><b>Subarna Biswas</b> joined the lab in 2025 as a Research Scientist. She is a trained and practicing surgeon, specializing in critical illness and emergency surgery. She received her MD from Stanford University and PhD in Epidemiology from USC in 2023. Her interests include investigating the genetic basis and evolutionary factors behind complex diseases, and the interplay between diet, environment, and adaptation. Originally from New Jersey, she enjoys exploring the quirky culture and diverse beauty of Los Angeles.</p>
					    </li>
					    
					    <li><p><h4>Lab Activities</h4></p>
					    <p>We don't just work hard, we play hard too! Click <a href="labactivities.html" target="_blank">here</a> for some pictures of the lab events and activities over the years!</p>

					    <p style="font-size:small">Image credit: modified from <a href="https://thenounproject.com/search/?q=group&i=2064675" target="_blank">"Group"</a> by Nawicon / CC </p>
					    </li>
					    
					    <li><p><h4>Lab Alumni</h4></p>
					    <p>All of the wonderful trainees, visitors, and colleagues that have contributed to our work!</p>
					    <p>
					    <b>Hanxiao Sun</b>, M.S. Biostatistics 2018-2019. Last known location: Postdoc at Memorial Sloan Kettering Cancer Center with Jian Carrot-Zhang<br />
					    <b>Meng Lin</b>, postdoc 2018-2020. Last known location: Senior Research Scientist in Biomedical Informatics at University of Colorado<br />
					    <b>Samuel Sommerer</b>, B.S. Quantitative Biology 2019-2020. Last known location: B.S. student in Computer Science at USC<br />
					    <b>Sydney Rashid</b>, B.S. Quantitative Biology 2019-2020. Last known location: B.S. student in Quantitative Biology at USC<br />
					    <b>Minhui Chen</b>, postdoc 2018-2021. Last known location: Assistant Professor at Virginia Commonwealth University, Virginia Institute for Psychiatric and Behavioral Genetics<br />
					    <b>Camellia Xinyue Rui</b>, M.S. Biostatistics 2020-2022. Last known location: Ph.D. student at USC<br/>
					    <b>Ying Chu Lo</b>, postdoc 2021-2023. Last known location: Research Computational Scientist II at Children's Hospital of Orange County<br /> 
					    <b>Christopher Simons</b>, B.S./M.S. Quantitative Biology 2021-2023. Last known location: Bioinformatician at Design Therapeutics<br/>
					    <b>Soyoung Jeon</b>, Ph.D. Cancer Biology and Genomics 2018-2023. Last known location: Computational Biologist at Genetech<br/>
					    <b>Caoqi Fan</b>, Ph.D. Computational Biology and Bioinformatics 2019-2023. Last known location: Bioinformatic scientist at Vibrant Therapeutics<br/>
					    <b>Echo Tang</b>, B.S. Quantitative Biology 2021-2023. Last known location: M.S. student in Public Health Data Sciences at USC<br/>
					    <b>Jordan Cahoon</b>, B.S. Computer Science 2021-2024. Last known location: Ph.D. student in Biomedical Data Science at Stanford<br/>
					    <b>Dorcus Malomane</b>, postdoc 2022-2025. Last known location: Bioinformatic scientist at The National Institute For Communicable Diseases, South Africa<br/>
					    </p>

					    <p style="font-size:small">Image credit: modified from <a href="https://thenounproject.com/search/?q=graduation&i=652526" target="_blank">"Graduation"</a> by Wilson Joseph / CC </p>
					    </li>
					    
					    <li><p><h4>Join Our Group</h4></p>
					    <p>We are always interested in talented and highly motivated individuals to join our team! We currently have space and funding for one or more postdoctoral fellows in genetic epidemiology and population genetics to join our group. Please check out <a href="resources.html" target="_blank">this page</a> for expectations and resources for lab members. See below for how to apply:</p>
				<!--	    <p><b>Postdoctoral Fellows through USC-Taiwan Postdoctoral Scholar Program:</b> We are looking for a postdoctoral fellow through the USC-Taiwan Partnered program. See <b>"Join.Us"</b> page for more details.</p> -->

					    
					    <p><b>Postdoctoral Fellows:</b> Highly motivated individuals are encouraged to contact Charleston Chiang for inquiries. In particular, there are openings for postdocs either interested in population genetics or in genetic epidemiology. Funding could be through one of multiple grant-funded projects, unrestricted funds, or through the <a href="https://uhcancercenter.org/education/trainees/t32-postdoctoral-fellowship" target="_blank">USC-UH Multiethnic Cohort T32 Postdoctoral Training Program</a>. Applicants ideally should have extensive exposure and experience in human medical and/or population genetics research and have analyzed large genetic datasets. Programming skills (e.g. python, perl, C, R, etc.) and proficiency in Unix-based computing environments are very desirable. Successful candidates may also develop projects broadly within the scope of the group's research interests.</p>
					    <p>Informal inquiries or applications (cover letter, CV, and contact information to at least two references) should be emailed to Charleston Chiang.</p>

					    
					    <p><b>Graduate Students:</b> The research group is recruiting prospective PhD students. Generally, doctoral students are enrolled through <a href="https://keck.usc.edu/pibbs/" target="_blank"> Program in Biological and Biomedical Sciences (PIBBS)</a>, <a href="https://dornsife.usc.edu/qcb/program-information/" target="_blank">Computational Biology and Bioinformatics (CBB)</a> or one of the <a href="http://publichealthsciences.usc.edu/phd-programs?hsCtaTracking=38ed720d-9e60-48cc-9982-85df7150d6dc%7C79817d43-fa7f-4f14-973e-0bbd62a83081" target="_blank">Public Health Sciences</a> programs (such as Biostatistics or Epidemiology) at USC. Email Charleston Chiang for inquiries.</p>

					    <p style="font-size:small">Image credit: <a href="https://thenounproject.com/term/unknown/55168/" target="_blank">"Unknown"</a> by Bybzee / CC BY</p>

					    </li>
					    
                                        </div> <!-- #slider -->
					
                                    </div>
                                        
                                </div>            

                            </section> <!-- #Team -->

			<!-- #Teaching -->
                            <section data-page-id="page-4" class="content js-content">
                                
				<header class="box box-black margin-b-20">
                                    <h2 class="box-text page-title-text">Teaching</h2>
                                </header>
                                
				<div class="content-text">
				  <h4><b>Courses</b></h4>
				  <p>PM534: Statistical Genetics (Fall 2021, 2022, 2024)<br />
				  Introduction to Medical Population Genetics<br />
				  <a href="https://www.dropbox.com/scl/fi/knlszrv4m3z9d8swwgzbj/Syllabus_CChiang_PM534_ScheduleOfClasses.pdf?rlkey=2k6oi1lhjgupehbhfg9hhwgp7&dl=0">syllabus (Fall 2021)</a><br />
				  <a href="https://www.dropbox.com/scl/fi/vjldymzlr6686zu0a18aq/Syllabus_CChiang_PM534_2022_FullSemester.pdf?rlkey=mnq7068bgykkdmzkp9p3datyd&dl=0">syllabus (Fall 2022)</a><br />
				  <a href="https://www.dropbox.com/scl/fi/cn55s9cazp5dty05gwwx5/Syllabus_PM534_20240822.pdf?rlkey=df8da9ergxao5yw4ug29zzh97&dl=0">syllabus (Fall 2024)</a><br />
				  </p>
				  <h4><b>Guest Lectures</b></h4>	
				  <p>BISC577: Computational Biology Laboratory (Fall 2019, Fall 2020, Fall 2021)<br />
				  "Genetics and Evolution": <a href="https://www.dropbox.com/scl/fi/2qucwj5k15qvw2hf0zq7l/BISC577A_Lecture_20200915.pptx?rlkey=k6963golpjeckchgzgllfbw3o&dl=0">slides (2020.09.15)</a><br />
				  (This is a huge slide deck! ~55Mb)</p>
				  
                                </div>            

                            </section> <!-- #Teaching -->

				<!-- #Contact -->
				<section data-page-id="page-5" class="content js-content">
                                
                                <header class="box box-black margin-b-20">
                                    <h2 class="box-text page-title-text">Contact Us</h2>
                                </header>
                                
				<div class="content-text">
				  <p>We are part of the <a href="http://keck.usc.edu/genetic-epidemiology-center/" target="_blank">Center for Genetic Epidemiology</a>, in the <a href="http://pphs.usc.edu/" target="_blank">Department of Population and Public Health Sciences</a> at <a href="http://keck.usc.edu/" target="_blank">USC Keck School of Medicine</a>, and jointly affiliated with the <a href="https://www.qcb-dornsife.usc.edu/" target="_blank">Department of Quantitative and Computational Biology</a> at USC Dornsife College of Letters, Arts, and Sciences.
				  </p>
				    <p>Charleston can be reached via email at:<br>
				    <b> charleston [dot] chiang [at] med [dot] usc [dot] edu</b></p>
<!--				    <p><b>We are always looking for talented and enthusiastic scientists to join our team!</b> If interested, please email a cover letter, CV, and (for postdoctoral applicants) contact information for two references to Charleston Chiang. See "Join.Us" tab for more details.</p>
				  -->
				    
				    <p>We are located in the Harlyne J. Norris Research Tower (NRT) on the Health Science Campus at USC</p>
				  
				    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3305.2881321738478!2d-118.20723008484106!3d34.06212738060329!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x80c2c5df4d307177%3A0x5f06b83977d9d185!2s1450+Biggy+St%2C+Los+Angeles%2C+CA+90033!5e0!3m2!1sen!2sus!4v1512006022128" width="520" height="400" frameborder="0" style="border:0" allowfullscreen></iframe>
				  
				    <p/><p>Mailing Address:<br />1450 Biggy Street<br />NRT-G511D<br />Los Angeles, CA 90033</p>
                                </div>            

                            </section> <!-- #Contact -->

				
			<!-- #Join Us -->
                            <section data-page-id="page-6" class="content js-content">
                                
				<header class="box box-black margin-b-20">
                                    <h2 class="box-text page-title-text">Join Us</h2>
                                </header>
                                
                                <div class="content-text">
				  <p><b>Postdoctoral Fellows:</b> We are always looking for talented and motivated individuals to join our group! Interested individuals are encouraged to contact Charleston Chiang for inquiries. There are multiple ways a postdoctoral fellow can be funded, through NIH grants in the lab, collaborative funds, or unrestricted funds. This means you have the stability as well as the flexibility to explore topics broadly of interest to the lab. Positions are generally for at least 2 years, with salary commensurate with experience and adjusted for cost of living in LA (definitely higher than NIH scale!). Applicants ideally should have extensive exposure and experience in human medical and/or population genetics research and have analyzed large genetic datasets. Programming skills (in at least one of, e.g., python, perl, C, R, etc.) and proficiency in Unix-based computing environments are very desirable. Successful candidates may also develop projects broadly within the scope of the group's research interests.</p>	
				  <p>Inquiries or applications (cover letter, CV, and contact information to at least two references) should be emailed to Charleston Chiang.</p>

<!--				  <p><b>T32 Postdoctoral Fellowship</b>: We currently have one opening for a postdoctoral fellow in genetic epidemiology. Applicants ideally should have extensive exposure and experience in human medical and/or population genetics research and have analyzed large genetic datasets. Programming skills (e.g. python, perl, C, R, etc.) and proficiency in Unix-based computing environments are very desirable. The position is funded for 2 years with possibility to extend for a 3rd year. Successful candidates may also develop projects broadly within the scope of the group's research interests using the Multiethnic Cohort, which consists of genotyping and sequencing data on up to 70K individuals from African American, Latino American, Japanese American, European American, and Native Hawaiian individuals.</p>
				  <p>More information can be found <a href="https://uhcancercenter.org/education/trainees/t32-postdoctoral-fellowship" target="_blank">here</a>. Informal inquiries should reach out to Charleston Chiang with a copy of CV.</p>
				  
				  <p><b>Postdoctoral Fellows through USC-Taiwan Postdoctoral Scholar Program</b>: Candidates are eligible if they received their Ph.D. no earlier than July 16, 2017 and no later than July 15, 2020. The degree must come from a Taiwanese institution, but Taiwanese nationals who have received degrees elsewhere are also eligible. The salary for postdoctoral fellow is $68,000 per year minus the cost of the benefits package. Fellow is expected to reside in the Los Angeles area during the academic year. More information on the program can be found <a href="https://postdocs.usc.edu/taiwan-usc-postdoctoral-fellowship-program/" target="_blank">here</a>. Online application is <a href="https://postdocs.usc.edu/taiwan-usc-application/" target="_blank">here</a>. Application deadline is July 15, 2020.</p>
				  <p>Informal inquiries should be emailed to Charleston Chiang, particularly if you would like to work together to develop a proposal for the application. An ideal candidate should have a recent Ph.D. in biology, genetics, computer science, bioinformatics, computational biology, or a related field. Proficiency in one or more programming languages (e.g. python, perl, C++, R, etc.) and in Unix-based computing environments is essential. Competitive applicants will also have extensive experience in conducting human genetics research and in analyzing large genetic datasets. Applicants can develop projects within the scope of the <a href="research.html" target="_blank">ongoing efforts</a> in the research group, and will have a chance to work on projected related to the ongoing collaboration with Taiwanese Biobank</p>
-->				  


				  <p><b>Graduate Students:</b>We are open to accepting doctoral and master students. Generally, doctoral students are enrolled through <a href="https://keck.usc.edu/pibbs/" target="_blank">Program in Biomedical and Biological Sciences (PIBBS)</a>, <a href="https://www.qcb-dornsife.usc.edu/phd" target="_blank">Computational Biology and Bioinformatics (CBB)</a>, or one of the programs in Population and Public Health Sciences (usually in <a href="https://keck.usc.edu/epidemiology-phd-program/" target="_blank">Epidemiology</a>) at USC. Email Charleston Chiang for inquiries.</p>

                                </div>            

                            </section> <!-- #Join Us -->


				<!-- contact FROM TEMPLATE DISABLED -->
				<!--
                            <section data-page-id="page-7" class="content js-content">
                                
                                <header class="box box-black margin-b-20">
                                    <h2 class="box-text page-title-text">Contact</h2>
                                </header>
                                
                                <div class="content-text">
                                    <p>In felis turpis, tincidunt a odio interdum, ornare interdum magna. Proin leo tortor, adipiscing et volutpat tincidunt, imperdiet sit amet purus.
                                    </p>
                                    <form action="index.html" method="post" class="contact-form">

                                        <div class="form-group col-xs-12 col-sm-12 col-md-6 col-lg-6 form-group-2-col-left">
                                            <input type="text" id="contact_name" name="contact_name" class="form-control" placeholder="Name" required/>
                                        </div>
                                        <div class="form-group col-xs-12 col-sm-12 col-md-6 col-lg-6 form-group-2-col-right">
                                            <input type="email" id="contact_email" name="contact_email" class="form-control" placeholder="Email" required/>
                                        </div>
                                        <div class="form-group">
                                            <input type="text" id="contact_subject" name="contact_subject" class="form-control" placeholder="Subject" required/>
                                        </div>
                                        <div class="form-group">
                                            <textarea id="contact_message" name="contact_message" class="form-control" rows="4" placeholder="Message" required></textarea>
                                        </div>
                                        <button type="submit" class="btn btn-primary submit-btn">Send</button>

                                    </form>                           
                                </div>            

                            </section> <!-- #contact -->
				-->

                        </div>
                    </div>

                </div>
            </div>        
                <!-- footer row -->
                <footer class="row footer js-footer">
                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">

                        <p class="copyright-text">Copyright &copy; 2017 Charleston Chiang<br />Click on "Chiang.Lab" to see the complete background</p>  

                    </div>                
                </footer>

            </div>  <!-- .container-fluid -->

        <div id="preloader">
            <div id="status">&nbsp;</div>
        </div><!-- /#preloader -->      
            
        <!-- load JS files -->
        <script src="js/jquery-1.11.3.min.js"></script> <!-- jQuery -->
        <script src="js/jquery.flexslider-min.js"></script> <!-- Flex Slider -->
        <script src="js/jquery.backstretch.min.js"></script> <!-- Backstretch http://srobbin.com/jquery-plugins/backstretch/ -->
        <script src="js/templatemo-script.js"></script> <!-- Templatemo scripts -->

	<!-- Start of StatCounter Code for Default Guide -->
	<script type="text/javascript">
	var sc_project=11576907; 
	var sc_invisible=1; 
	var sc_security="4cde62bc"; 
	var scJsHost = (("https:" == document.location.protocol) ?
	"https://secure." : "http://www.");
	document.write("<sc"+"ript type='text/javascript' src='" +
	scJsHost+
	"statcounter.com/counter/counter.js'></"+"script>");
	</script>
	<noscript><div class="statcounter"><a title="Web Analytics
	  Made Easy - StatCounter" href="http://statcounter.com/"
	  target="_blank"><img class="statcounter"
	  src="//c.statcounter.com/11576907/0/4cde62bc/1/" alt="Web
	  Analytics Made Easy - StatCounter"></a></div></noscript>
	<!-- End of StatCounter Code for Default Guide -->

	
    </body>
</html>