<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
	</head>
	<body>

		<!-- Header -->
			<section id="header">
				<header>
					<span class="image avatar"><img src="img/Regular_Use_Shield_BlackOnGold.jpg" alt="" /></span>
					<h1 id="logo">Research at Chiang Lab</h1>
					<p>We are interested in addressing questions
					at the intersection of <b><i>medical</i></b> and <b><i>population</i></b> genetics in humans</p>
				</header>
				<nav id="nav">
					<ul>
						<li><a href="#research" class="active">Research</a></li>
						<li><a href="#data">Data/Softwares</a></li>
						<li><a href="#publication">Publications</a></li>
						<!-- <li><a href="#four">Contact</a></li> -->
						<li><a href="index.html">Home</a></li>
					</ul>
				</nav>
					<!--
					<footer>
					<ul class="icons">
						<li><a href="#" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="#" class="icon fa-facebook"><span class="label">Facebook</span></a></li>
						<li><a href="#" class="icon fa-instagram"><span class="label">Instagram</span></a></li>
						<li><a href="#" class="icon fa-github"><span class="label">Github</span></a></li>
						<li><a href="#" class="icon fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					</footer>
					-->
			</section>

		<!-- Wrapper -->
			<div id="wrapper">

			  <h1 id="usclogo"><a href="https://www.usc.edu">usc</a></h1>
					  
			  <!-- Main -->
					<div id="main">

					  <!-- Research -->
					  <section id="research">
					        <div class="container">
						       <h3>Research</h3>
						  <p>The overarching theme of our research group is to <b>use genetic approaches, both empirical and theoretical, to understand how evolutionary forces shaped the genetic architecture of complex traits within and between populations</b>. To this end, we have been involved in a number of past and ongoing medical genetics studies in mapping genetic loci underlying human complex traits. We are also continually interested in investigating the evolutionary forces, namely demography and selection, that shaped the pattern of genetic variability and phenotypic distribution. We are particularly interested in diverse, global human populations and our successes result from collaborating with innovative colleagues and thriving in resourceful consortiums.</p>
						  <p><br />The focus of the group mainly lies in the following areas:</p>
						  <h4><b> Genetic basis of complex traits in diverse populations</b></h4>
						  <p><span class="image left close">
						  <img src="img/finland.jpg" alt="" height=180 /></span>
						  <span class="image left"><img src="img/NG_201511.gif" alt="" height=180 /></span>
						  Our group has led past and ongoing genetic studies in mapping genetic loci underlying human complex phenotypes such as anthropometric and cardiometabolic traits. Our focus has always been in studying diverse populations with special demographic history. For example, we conducted one of the first GWAS of African-ancestry individuals for anthropometric traits (<b>PLoS Genet. 2010</b>; <b>Hum. Mol. Genet. 2010</b>). More recently, we leveraged the special population histories to map novel rare variant associated with height in Sardinians (<b>Nat. Genet. 2015</b>), as well as discovering scores of novel rare alleles associated with quantitative cardiometabolic traits in Northern and Eastern Finns (<b>Nature 2019</b>).</p>
						  
<p>We are currently focusing on understanding the genetic architecture of complex traits in diverse, understudied, and underserevd populations. There is a huge opportunity to leverage the evolutionary history of diverse populations to design and interpret genetic mapping studies (<b>Front. Genet. 2021</b>). However, we have shown that for diverse populations such as the Native Hawaiians, the lack of representation in genomic resources (e.g. a Polynesian haplotype reference panel for imputation) has been prohibitive to research (<b>Hum. Mol. Genet. 2020</b>). Therefore even though we know that epidemiologically Native Hawaiians have elevated risk for certain complex diseases, and we can show that this disease risk correlat with proportion of Polynesian genetic ancestry (but note that ancestry captures also non-genetic effects; <b>PLoS Genet. 2021</b>), we know very little of population-specific alleles that impacts disproportionately in these populations and thereby contributing to health disparities between populations. We are currently generating whole-genome sequencing data and constructing the genomic resources necessary to accelerate research in Polynesian populations. We are also forming a consortium of Pacific Island populations, including Native Hawaiians, Samoans, and other Pacific Islanders. In aggregate we will reach >13,000 Pacific Islanders with the goals of identifying more population-specific variants such as CREBRF (Minster et al. Nat. Genet. 2016) across Pacific Islanders. We are also trying to strengthen our interactions and engagement with the Native community, through interactions with the Native Hawaiian Community Advisory Board estsablished by the University of Hawaii Cancer Center, as well as through planned focus group to understand the community concerns with participating in genomic research.</p>

<p>In addition to population-specific studies, we are also taking an multiethnic approach to illuminate the complex genetic architecture of quantitative traits and diseases. For example, we were part a multi-ethnic consortium that identified 71 novel loci associated with hematological traits that are not found in European ancestry populations (<b>Cell 2020</b>). Moreover, We have performed multiethnic meta-analysis in a California-based cohort for Acute Lymphoblastic Leukemia (ALL) and identified three potential novel loci (<b>Leukemia 2022</b>). We are co-leading the genetic analysis of Latino and international multi-ethnic consortium to detect additional variants associated with ALL. Together through the Childhood Leukemia International Consortium (CLIC), we have accrued >10,000 cases and >100,000 controls in multiple ethnicities for genetic mapping and fine-mapping.</p>
						  <br />
						  <h4><b> Fine-scale structure and demographic history of global populations</b></h4>
<p><span class="image right"><img src="img/NG_201810.png" alt="" height=180 /></span>
<span class="image right close"><img src="img/mediteems.jpg" alt="" height=180 /></span>
One burgeoning movement in medicine is the incorporation of evolutionary principles. Medicine is classically concerned with only the most proximate causes of dysfunction and views diseases as defects of a once perfect machine. In contrast, evolutionary medicine strives to view diseases as the consequence of trade-offs and vulnerabilities due to interactions with our past environments. However, we need to first understand the demographic history and population structure of the population of interest, which in turns guide our design of the genetic studies to understand the genetic architecture and evolutionary courses of complex traits.</p>
<p><span class="image left"><img src="img/chinapca.jpg" alt="" height=180 /></span>
To this end, we have investigated the fine-scale structure and history of human populations known to have a unique past or relatively understudied. We found geographical features that led to the genetic barrier and population structure in Sardinia (<b>Nat. Genet. 2018</b>). We conducted one of the first examinations of population structure in Han Chinese living in Taiwan (<b>Hum. Mol. Genet. 2016</b>), and identified additional geographical axis using whole genome sequence data in Han Chinese living in China (<b>Mol. Biol. Evol. 2018</b>). We also characterized the fine-scale structure and the consequent enrichment of deleterious variation in Northern and Eastern Finland, thereby enabled unprecedented power to map rare variant associations (<b>Nature 2019</b>). Future goals will continue to investigate understudied diverse populations, such as the Native Hawaiians in the Multiethnic Cohort, Han Chinese and Aboriginal Taiwanese in Taiwan Biobank, or East Asians living in the United States. We will aim to understand in finer details of the population structure and admixture history of these populations to shed light on how differential susceptibility to diseases across populations may be rooted in their demographic histories.</p>

<p>We are also developing innovative methods. Thanks to recent advancements that allows scalable inference of Ancestral Recombination Graphs genome-wide (Spiedel et al. Nat. Genet. 2018; Kelleher et al. Nat. Genet. 2018), we can approximate well the underlying genealogy that relates every one in the sample. Based on these sequences of genealogical trees, we proposed a novel metric of genetic relationship matrix (GRM) that computes the expected relatedness between pairs of individual given the genealogies. We found this "expected GRM" or eGRM is more sensitive to recent population structure, and has the flexibility and potential to detect the varying nature of structure over time (<b>AJHG 2022</b>).</p> 
<br />
						  <h4><b> Natural Selection and Polygenic Adaptation </b></h4>
<p><span class="image left"><img src="img/ng.3403-F3.jpg" alt="" height=180 /></span>
Demographic processes such as long-term isolation and bottleneck are not the only forces that shape the pattern of variation in our genome. Natural selection is another important factor that contributes to our genomic pattern of variation and distribution of complex traits. We have conducted genome-wide scans for signals of selection in human populations (<b>Mol. Biol. Evol. 2018</b>), as well as focusing on the polygenic signature of adaptation of copmlex traits, namely height. Using genome-wide summary statistics from GWAS we were the first to suggest that differentiation of adult human height across Europe is driven by polygenic adaptation (<b>Nat. Genet. 2012</b>). We later also demonstrated that the significant shorter stature in Sardinia is also driven by polygenic adaptation (<b>Nat. Genet. 2015</b>). Our study in human height stimulated the development of many methods aimed to detect such signatures in other complex traits as well as continued and ongoing discussion on interpreting polygenic scores from large scale GWAS (<b>eLife 2019</b>; also see Berg et al. eLife 2019). In light of the new insight that subtle uncorrected population structure could bias analysis of polygenic adaptation, we now leverage large-scale biobank data from distant, unrelated population such as the Biobank of Japan (BBJ) to unbiasly assess polygenic adaptation in Europeans. We found that differences in height between Sardinians and mainland Europeans are robustly attributed to polygenic adaptation, and we continue to find evidence consistent with natural selection driving the differences in height across mainland Europe, although the effect is weaker than previously believed (<b>AJHG 2020</b>). Using the same robust ascertainment approach, we also found that height-associated loci remain exceptionally differentiated across global populations, again supporting some form of natural selection that results in differentiation of height across populations (<b>EJHG 2021</b>)</p>
<p>Moving forward, we will continue to investigate these imprints left in our genome by Natural Selection, and integrate external non-genetic information to contextualize these findings, which may reveal cases where increased susceptibility of certain traits may be a trade-off due to an evolutionary benefit in our past. For example, we will examine if elevated risk of childhood leukemia found in Latino Americans is due to selection of specific alleles associated with protection against infection in their past history</p>

						</div>
						</section>

					  <!-- Data resources -->	
					  <section id="data">
					        <div class="container">
						  <h3>Datasets and Software Resources</h3>
						  <p><b>Data Release (Chiang et al. Mol. Bio. Evol. 2018)</b></p>
						  <ul>
						    <li>List of > 82K novel variants found with MAF > 5% in > 10,000 Whole genome sequenced Han Chinese but not found in 1000G CHB+CHS populations. When also found in ExAC or GNOMAD East Asians, the frequencies are annotated. All coordinates in hg19. <a href="rsch/novel.variants.sorted.maf5p.annotated.ExAC.GNOMAD.txt">file</a></li>
						    <li>Allele frequencies stratified by provinces can be found at GGV (with google or twitter login, under "Converge"). <a href="https://popgen.uchicago.edu/dev-integrated/">link</a></li>
						    <li>Sample birth location of the Han Chinese used in PCA. <a href="rsch/han.chinese.origins.txt">file</a></li></br>
						  </ul>

						  <p><b>Expected Genetic Relationship Matrix (Fan et al. AJHG 2022)</b>: constructs the expected GRM given inferred ancestral recombination graphs. <a href="https://github.com/Ephraim-usc/egrm" target="_blank">github</a></p>

						  <p><b>triple-liftOver (Sheng et al. bioRxiv 2022)</b>: identifies variants mapping into a region inverted between genome builds to avoid downstream analysis problem such as imputation. <a href="https://github.com/GraceSheng/triple-liftOver" target="_blank">github</a></p>
						  
						  <!-- <p> Please check back here for updates in the future!</p> -->
						</div>
						</section>

					  <section id="publication">
					        <div class="container">
						       <h3>Selected Publications</h3>
						  <p> You can check <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=charleston+chiang" target="_blank">pubmed</a> or <a href="https://scholar.google.com/citations?user=qYy0_YwAAAAJ&hl=en" target="_blank">google scholar</a> for the complete list of publication from the lab.</p>
						  <p> Group members are <b>bolded</b>; *<sup>, #</sup> denote equal contributions</p>
						  <ul>
						    <li>Sheng X, Xia L, Conti DV, Haiman CA, Kachuri L, <b>Chiang CWK</b>. "Inverted genomic regions between reference genome builds in humans impact imputation accuracy and decrease the power of association testing." <b>bioRxiv</b>. 2022. <a href="https://www.biorxiv.org/content/10.1101/2022.02.19.481174v2">link</a></li><br />
						    <li><b>Fan C</b>, Mancuso N*, <b>Chiang CWK*</b>. “A genealogical estimate of genetic relationships.” <b>Am J Hum Genet</b>. 2022 Apr 6:S0002-9297(22)00112-4. doi: 10.1016/j.ajhg.2022.03.016. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(22)00112-4">link</a></li><br />
						    <li><b>Jeon S</b>, de Smith AJ, Li S, <b>Chen M</b>, <b>Chan TF</b>, Muskens IS, Morimoto LM, Dewan AT, Mancuso N, Metayer C, Ma X, Wiemels JL, <b>Chiang CWK</b>. “Genome-wide trans-ethnic meta-analysis identifies novel susceptibility loci for childhood acute lymphoblastic leukemia.” <b>Leukemia</b>. 2022 Mar;36(3):865-868. doi: 10.1038/s41375-021-01465-1. Epub 2021 Nov 8. <a href="https://pubmed.ncbi.nlm.nih.gov/34750507/">link</a></li><br />
						    <li><b>Chiang CWK</b>. “The opportunities and challenges of integrating population histories into genetic studies of diverse populations: a motivating example from Native Hawaiians.” <b>Front Genet</b>. 2021 Sep 27;12:643883. doi: 10.3389/fgene.2021.643883. <a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.643883/full">link</a>.</li><br />
						    <li><b>Chen M</b> and <b>Chiang CWK</b>. "Allele frequency differentiation at height-associated SNPs among contin-ental human populations.” <b>Eur J Hum Genet</b>. 2021 Oct;29(10):1542-1548. doi: 10.1038/s41431-021-00938-2. Epub 2021 Jul 15. <a href="https://pubmed.ncbi.nlm.nih.gov/34267339/">link</a>.</li><br />
						    <li><b>Sun H*</b>, <b>Lin M*</b>, Russell EM, Minster RL, <b>Chan TF</b>, <b>Dinh BL</b>, Naseri T, Reupena MS, Lum-Jones A, OLaGA Study Group, Cheng I, Wilkens LR, Le Marchand L, Haiman CA, <b>Chiang CWK</b>. “The impact of global and local Polynesian genetic ancestry on complex traits in Native Hawaiians.” <b>PLoS Genet</b>. 2021 Feb 11;17(2):e1009273. doi: 10.1371/journal.pgen.1009273. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009273">link</a>.</li><br />
						    <li>Chen MH, Raffield LM, Mousas A, Sakaue S, ..., <b>Chiang CWK</b>, Li B, Loos RJF, Astle WJ, Evangelou E, Sankaran VG, Okada Y, Soranzo N, Johnson AD, Reiner AP, Auer PL, Lettre G. "Trans-ethnic and ancestry-specific blood cell genetics in 746,667 individuals from 5 global populations." <b>Cell</b>. 2020 Sep 3;182(5):1198-1213.e14. <a href="https://pubmed.ncbi.nlm.nih.gov/32888493/">link</a>.</li><br />
						    <li><b>Lin M</b>, Caberto C, Wan P, Li Y, Lum-Jones A, Tiirikainen M, Pooler L, Nakamura B, Sheng X, Porcel J, Lim U, Setiawan VW, Le Marchand L, Wilkens LR, Haiman CA, Cheng I, <b>Chiang CWK</b>. “Population specific reference panels are crucial for the genetic analyses of Native Hawaiians: an example of the CREBRF locus.” <b>Hum Mol Genet</b>. 2020 Aug 3;29(13):2275-2284.  doi: 10.1093/hmg/ddaa083. <a href="https://pubmed.ncbi.nlm.nih.gov/32491157/">link</a>.</li><br />
						    <li><b>Chen M</b>, Sidore C, Akiyama M, Ishigaki K, Kamatani Y, Schlessinger D, Cucca F, Okada Y, <b>Chiang CWK</b>. “Evidence of polygenic adaptation at height-associated loci in Sardinians using Biobank Japan.” <b>Am J Hum Genet</b>. 2020 Jul 2;107(1):60-71. doi: 10.1016/j.ajhg.2020.05.014. Epub 2020 Jun 12. <a href="https://pubmed.ncbi.nlm.nih.gov/32533944/">link</a></li><br />
						    <li>Locke AE*, Steinberg KM*, <b>Chiang CWK*</b>, Service S, Havulinna A, Stell L, Pirinen M, Abel HJ, Chiang CC, Fulton RS, Jackson AU, Kang CJ, Kanchi KL, Koboldt DC, Larson DE, Nelson J, Nicholas TJ, Pietila A, Ramensky V, Ray D, Scott LJ, Stringham HM, Vangipurapu J, Welch R, Yajnik P, Yin X, Eriksson JG, Ala-Korpela M, Jarvelin MR, Manniko M, Laivouri H, FinnGen Project, Dutcher SK, Stitziel NO, Wilson RK, Hall IM, Sabatti C, Palotie A, Salomaa V, Laakso M, Ripatti S, Boehnke M, Freimer NB. “Exome sequencing of Finnish isolates enhances rare-variant association power.” <b>Nature</b>. 2019 Aug;572(7769):323-328. doi: 10.1038/s41586-019-1457-z. Epub 2019 Jul 31. <a href="https://www.nature.com/articles/s41586-019-1457-z">link</a></li><br />
						    <li>Sohail M, Maier RM, Ganna A, Bloemendal A, Martin AR, Turchin MC, <b>Chiang CWK</b>, Hirschhorn J, Daly MJ, Patterson N, Neale B, Mathieson I, Reich D, Sunyaev SR. "Polygenic adaptation on height is overestimated due to uncorrected stratification in genome-wide association studies." <b>Elife</b>. 2019 Mar 21;8. pii: e39702. doi: 10.7554/eLife.39702. <a href="https://elifesciences.org/articles/39702">link</a></li><br />
						    <li><b>Chiang CWK</b>, Mangul S, Robles C, Sankararaman S. "A comprehensive map of genetic variation in the world's largest ethnic group - Han Chinese." <b>Mol Bio Evol</b>. 2018 Nov 1;35(11):2736-2750. doi: 10.1093/molbev/msy170. Epub 2018 Aug 30. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30169787">link</a></li><br />
						    <li><b>Chiang CWK</b>, Marcus J, Sidore C,Al-asadi H, Biddanda A, Zoledziewska M, Pistis G, Steri M, Lohmueller K, Abecasis G, Schlessinger D, Cucca F, Novembre J. "Genomic History of the Sardinian Population" <b>Nat Genet</b>. 2018 Oct;50(10):1426-1434. Epub 2018 Sep 17. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30224645">link</a></li><br />
						    <li>Chen CH, Yang JH, <b><big>Chiang CWK</big></b>, Hsiung CN, Wu PE, Chang LC, Chang J, Song IW, Yang SL, Chen YT, Liu FT, Shen CY. "Population structure of Han Chinese in the modern Taiwanese population based on 10,000 participants in the Taiwan Biobank project." <b>Hum Mol Genet</b>. 2016 Dec 15;25(24):5321-5331.<a href="https://www.ncbi.nlm.nih.gov/pubmed/27798100">link</a></li><br />
						    <li>Zoledziewska M*, Sidore C*, <b>Chiang CWK*</b>, Sanna S*, Mulas A, Steri M, Busonero F, Marcus JH, Marongiu M, Maschio A, Ortega Del Vecchyo VD, Floris M, Meloni A, Delitala A, Meloni A, Delitala A, Concas MP, Murgia F, Biino G, Vaccargiu S, Nagaraja R, Lohmueller KE; UK10K Consortium, Timpson NJ, Soranzo N, Tachmazidou I, Dedoussis G, Zeggini E; Understanding Society Scientific Group, Uzzau S, Jones C, Lyons R, Angius A, Abecasis GR<sup>#</sup>, Novembre J<sup>#</sup>, Schlessinger D<sup>#</sup>, Cucca F<sup>#</sup>. "Height-reducing variants and selection for short stature in Sardinia." <b>Nat Genet</b>. 2015 Nov; 47(11):1352-6. doi: 10.1038/ng.3403. Epub 2015 Sep 14. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26366551">link</a></li><br />
						    <li>Turchin MC*, <b>Chiang CWK*</b>, Palmer CD, Sankararaman S, Reich D, Genetic Investigation of ANthropometric Traits (GIANT) Consortium, Hirschhorn JN. "Evidence of widespread selection on standing variation in Europe at height-associated SNPs." <b>Nat Genet</b>. 2012 Sep;44(9):1015-9. doi: 10.1038/ng.2368. Epub 2012 Aug 19. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22902787">link</a></li><br />
						    <li><b>Chiang CWK</b>, Liu CT, Lettre G, Lange LA, Jorgensen NW, Keating BJ, Vedantam S, Nock NL, Franceschini N, Reiner AP, Demerath EW, Boerwinkle E, Rotter JI, Wilson JG, North KE, Papanicolaou GJ, Cupples LA, Genetic Investigation of ANthropometric Traits (GIANT) consortium, Murabito JM, Hirschhorn JN. "Ultraconserved elements in the human genome: association and transmission analyses of highly constrained single-nucleotide polymorphisms." <b>Genetics</b>. 2012 Sep;192(1):253-66. Epub 2012 Jun 19. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22714408">link</a></li><br />
						    <li>Kang SJ*, <b>Chiang CWK*</b>, Palmer CD*, Tayo BO, Lettre G, Butler JL, Hackett R, Adeyemo AA, Guiducci C, berzins I, Nguyen TT, Feng T, Luke A, Shriner D, Ardlie K, Rotimi C, Wilks R, Forrester T, McKenzie CA, Lyon HN, Cooper RS, Zhu X<sup>#</sup>, Hirschhorn JN<sup>#</sup>. "Genome wide association of anthropometric traits in African and African derived populations." <b>Hum Mol Genet</b>. 2010 Jul 1;19(13):2725-38. Epub 2010 Apr 16. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20400458">link</a></li><br />
						    <li><b>Chiang CWK</b>, Gajdos ZKZ, Korn JM, Kuruvilla FG, Butler JL, Hackett R, Guiducci C, Nguyen TT, Wilks R, Forrester T, Haiman CA, Henderson KD, Le Marchand L, Henderson BE, Palmert MR, McKenzie CA, Lyon HN, Cooper RS, Zhu X, Hirschhorn JN. "Rapid assessment of genetic ancestry inpopulatiosn of unknown origin by genome-wide genotyping of pooled samples." <b>PLoS Genetics</b>. 2010;6(3):e1000866. Epub 2010 Mar 5. <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000866">link</a></li><br />
						  </ul>
						</div>
						</section>
						

						
						<!-- One -->
						<!--
						<section id="one">
								<div class="container">
									<header class="major">
										<h2>Read Only</h2>
										<p>Just an incredibly simple responsive site<br />
										template freebie by <a href="http://html5up.net">HTML5 UP</a>.</p>
									</header>
									<p>Faucibus sed lobortis aliquam lorem blandit. Lorem eu nunc metus col. Commodo id in arcu ante lorem ipsum sed accumsan erat praesent faucibus commodo ac mi lacus. Adipiscing mi ac commodo. Vis aliquet tortor ultricies non ante erat nunc integer eu ante ornare amet commetus vestibulum blandit integer in curae ac faucibus integer non. Adipiscing cubilia elementum.</p>
								</div>
							</section>
								-->
								
						<!-- Two -->
						<!--
							<section id="two">
								<div class="container">
									<h3>Things I Can Do</h3>
									<p>Integer eu ante ornare amet commetus vestibulum blandit integer in curae ac faucibus integer non. Adipiscing cubilia elementum integer lorem ipsum dolor sit amet.</p>
									<ul class="feature-icons">
										<li class="fa-code">Write all the code</li>
										<li class="fa-cubes">Stack small boxes</li>
										<li class="fa-book">Read books and stuff</li>
										<li class="fa-coffee">Drink much coffee</li>
										<li class="fa-bolt">Lightning bolt</li>
										<li class="fa-users">Shadow clone technique</li>
									</ul>
								</div>
							</section>
								-->
								
								
						<!-- Three -->
						<!--		
							<section id="three">
								<div class="container">
									<h3>A Few Accomplishments</h3>
									<p>Integer eu ante ornare amet commetus vestibulum blandit integer in curae ac faucibus integer non. Adipiscing cubilia elementum integer. Integer eu ante ornare amet commetus.</p>
									<div class="features">
										<article>
											<a href="#" class="image"><img src="images/pic01.jpg" alt="" /></a>
											<div class="inner">
												<h4>Possibly broke spacetime</h4>
												<p>Integer eu ante ornare amet commetus vestibulum blandit integer in curae ac faucibus integer adipiscing ornare amet.</p>
											</div>
										</article>
										<article>
											<a href="#" class="image"><img src="images/pic02.jpg" alt="" /></a>
											<div class="inner">
												<h4>Terraformed a small moon</h4>
												<p>Integer eu ante ornare amet commetus vestibulum blandit integer in curae ac faucibus integer adipiscing ornare amet.</p>
											</div>
										</article>
										<article>
											<a href="#" class="image"><img src="images/pic03.jpg" alt="" /></a>
											<div class="inner">
												<h4>Snapped dark matter in the wild</h4>
												<p>Integer eu ante ornare amet commetus vestibulum blandit integer in curae ac faucibus integer adipiscing ornare amet.</p>
											</div>
										</article>
									</div>
								</div>
							</section>
								-->

								
						<!-- Four -->
						<!--
								<section id="four">
								<div class="container">
									<h3>Contact Me</h3>
									<p>Integer eu ante ornare amet commetus vestibulum blandit integer in curae ac faucibus integer non. Adipiscing cubilia elementum integer. Integer eu ante ornare amet commetus.</p>
									<form method="post" action="#">
										<div class="row uniform">
											<div class="6u 12u(xsmall)"><input type="text" name="name" id="name" placeholder="Name" /></div>
											<div class="6u 12u(xsmall)"><input type="email" name="email" id="email" placeholder="Email" /></div>
										</div>
										<div class="row uniform">
											<div class="12u"><input type="text" name="subject" id="subject" placeholder="Subject" /></div>
										</div>
										<div class="row uniform">
											<div class="12u"><textarea name="message" id="message" placeholder="Message" rows="6"></textarea></div>
										</div>
										<div class="row uniform">
											<div class="12u">
												<ul class="actions">
													<li><input type="submit" class="special" value="Send Message" /></li>
													<li><input type="reset" value="Reset Form" /></li>
												</ul>
											</div>
										</div>
									</form>
								</div>
							</section>
								-->


								
						<!-- Five -->
								<!--
							<section id="five">
								<div class="container">
									<h3>Elements</h3>

									<section>
										<h4>Text</h4>
										<p>This is <b>bold</b> and this is <strong>strong</strong>. This is <i>italic</i> and this is <em>emphasized</em>.
										This is <sup>superscript</sup> text and this is <sub>subscript</sub> text.
										This is <u>underlined</u> and this is code: <code>for (;;) { ... }</code>. Finally, <a href="#">this is a link</a>.</p>
										<hr />
										<header>
											<h4>Heading with a Subtitle</h4>
											<p>Lorem ipsum dolor sit amet nullam id egestas urna aliquam</p>
										</header>
										<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat eu sed ante lacinia sapien lorem accumsan varius montes viverra nibh in adipiscing blandit tempus accumsan.</p>
										<header>
											<h5>Heading with a Subtitle</h5>
											<p>Lorem ipsum dolor sit amet nullam id egestas urna aliquam</p>
										</header>
										<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat eu sed ante lacinia sapien lorem accumsan varius montes viverra nibh in adipiscing blandit tempus accumsan.</p>
										<hr />
										<h2>Heading Level 2</h2>
										<h3>Heading Level 3</h3>
										<h4>Heading Level 4</h4>
										<h5>Heading Level 5</h5>
										<h6>Heading Level 6</h6>
										<hr />
										<h5>Blockquote</h5>
										<blockquote>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan faucibus. Vestibulum ante ipsum primis in faucibus lorem ipsum dolor sit amet nullam adipiscing eu felis.</blockquote>
										<h5>Preformatted</h5>
										<pre><code>i = 0;

	while (!deck.isInOrder()) {
	    print 'Iteration ' + i;
	    deck.shuffle();
	    i++;
	}

	print 'It took ' + i + ' iterations to sort the deck.';</code></pre>
									</section>

									<section>
										<h4>Lists</h4>
										<div class="row">
											<div class="6u 12u(xsmall)">
												<h5>Unordered</h5>
												<ul>
													<li>Dolor pulvinar etiam magna etiam.</li>
													<li>Sagittis adipiscing lorem eleifend.</li>
													<li>Felis enim feugiat dolore viverra.</li>
												</ul>
												<h5>Alternate</h5>
												<ul class="alt">
													<li>Dolor pulvinar etiam magna etiam.</li>
													<li>Sagittis adipiscing lorem eleifend.</li>
													<li>Felis enim feugiat dolore viverra.</li>
												</ul>
											</div>
											<div class="6u 12u(xsmall)">
												<h5>Ordered</h5>
												<ol>
													<li>Dolor pulvinar etiam magna etiam.</li>
													<li>Etiam vel felis at lorem sed viverra.</li>
													<li>Felis enim feugiat dolore viverra.</li>
													<li>Dolor pulvinar etiam magna etiam.</li>
													<li>Etiam vel felis at lorem sed viverra.</li>
													<li>Felis enim feugiat dolore viverra.</li>
												</ol>
												<h5>Icons</h5>
												<ul class="icons">
													<li><a href="#" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
													<li><a href="#" class="icon fa-facebook"><span class="label">Facebook</span></a></li>
													<li><a href="#" class="icon fa-instagram"><span class="label">Instagram</span></a></li>
													<li><a href="#" class="icon fa-github"><span class="label">Github</span></a></li>
													<li><a href="#" class="icon fa-dribbble"><span class="label">Dribbble</span></a></li>
													<li><a href="#" class="icon fa-tumblr"><span class="label">Tumblr</span></a></li>
												</ul>
											</div>
										</div>
										<h5>Actions</h5>
										<ul class="actions">
											<li><a href="#" class="button special">Default</a></li>
											<li><a href="#" class="button">Default</a></li>
											<li><a href="#" class="button alt">Default</a></li>
										</ul>
										<ul class="actions small">
											<li><a href="#" class="button special small">Small</a></li>
											<li><a href="#" class="button small">Small</a></li>
											<li><a href="#" class="button alt small">Small</a></li>
										</ul>
										<div class="row">
											<div class="3u 6u(medium) 12u$(xsmall)">
												<ul class="actions vertical">
													<li><a href="#" class="button special">Default</a></li>
													<li><a href="#" class="button">Default</a></li>
													<li><a href="#" class="button alt">Default</a></li>
												</ul>
											</div>
											<div class="3u 6u$(medium) 12u$(xsmall)">
												<ul class="actions vertical small">
													<li><a href="#" class="button special small">Small</a></li>
													<li><a href="#" class="button small">Small</a></li>
													<li><a href="#" class="button alt small">Small</a></li>
												</ul>
											</div>
											<div class="3u 6u(medium) 12u$(xsmall)">
												<ul class="actions vertical">
													<li><a href="#" class="button special fit">Default</a></li>
													<li><a href="#" class="button fit">Default</a></li>
													<li><a href="#" class="button alt fit">Default</a></li>
												</ul>
											</div>
											<div class="3u 6u$(medium) 12u$(xsmall)">
												<ul class="actions vertical small">
													<li><a href="#" class="button special small fit">Small</a></li>
													<li><a href="#" class="button small fit">Small</a></li>
													<li><a href="#" class="button alt small fit">Small</a></li>
												</ul>
											</div>
										</div>
									</section>

									<section>
										<h4>Table</h4>
										<h5>Default</h5>
										<div class="table-wrapper">
											<table>
												<thead>
													<tr>
														<th>Name</th>
														<th>Description</th>
														<th>Price</th>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>Item One</td>
														<td>Ante turpis integer aliquet porttitor.</td>
														<td>29.99</td>
													</tr>
													<tr>
														<td>Item Two</td>
														<td>Vis ac commodo adipiscing arcu aliquet.</td>
														<td>19.99</td>
													</tr>
													<tr>
														<td>Item Three</td>
														<td> Morbi faucibus arcu accumsan lorem.</td>
														<td>29.99</td>
													</tr>
													<tr>
														<td>Item Four</td>
														<td>Vitae integer tempus condimentum.</td>
														<td>19.99</td>
													</tr>
													<tr>
														<td>Item Five</td>
														<td>Ante turpis integer aliquet porttitor.</td>
														<td>29.99</td>
													</tr>
												</tbody>
												<tfoot>
													<tr>
														<td colspan="2"></td>
														<td>100.00</td>
													</tr>
												</tfoot>
											</table>
										</div>

										<h5>Alternate</h5>
										<div class="table-wrapper">
											<table class="alt">
												<thead>
													<tr>
														<th>Name</th>
														<th>Description</th>
														<th>Price</th>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>Item One</td>
														<td>Ante turpis integer aliquet porttitor.</td>
														<td>29.99</td>
													</tr>
													<tr>
														<td>Item Two</td>
														<td>Vis ac commodo adipiscing arcu aliquet.</td>
														<td>19.99</td>
													</tr>
													<tr>
														<td>Item Three</td>
														<td> Morbi faucibus arcu accumsan lorem.</td>
														<td>29.99</td>
													</tr>
													<tr>
														<td>Item Four</td>
														<td>Vitae integer tempus condimentum.</td>
														<td>19.99</td>
													</tr>
													<tr>
														<td>Item Five</td>
														<td>Ante turpis integer aliquet porttitor.</td>
														<td>29.99</td>
													</tr>
												</tbody>
												<tfoot>
													<tr>
														<td colspan="2"></td>
														<td>100.00</td>
													</tr>
												</tfoot>
											</table>
										</div>
									</section>

									<section>
										<h4>Buttons</h4>
										<ul class="actions">
											<li><a href="#" class="button special">Special</a></li>
											<li><a href="#" class="button">Default</a></li>
											<li><a href="#" class="button alt">Alternate</a></li>
										</ul>
										<ul class="actions">
											<li><a href="#" class="button special big">Big</a></li>
											<li><a href="#" class="button">Default</a></li>
											<li><a href="#" class="button alt small">Small</a></li>
										</ul>
										<ul class="actions fit">
											<li><a href="#" class="button special fit">Fit</a></li>
											<li><a href="#" class="button fit">Fit</a></li>
											<li><a href="#" class="button alt fit">Fit</a></li>
										</ul>
										<ul class="actions fit small">
											<li><a href="#" class="button special fit small">Fit + Small</a></li>
											<li><a href="#" class="button fit small">Fit + Small</a></li>
											<li><a href="#" class="button alt fit small">Fit + Small</a></li>
										</ul>
										<ul class="actions">
											<li><a href="#" class="button special icon fa-download">Icon</a></li>
											<li><a href="#" class="button icon fa-download">Icon</a></li>
											<li><a href="#" class="button alt icon fa-check">Icon</a></li>
										</ul>
										<ul class="actions">
											<li><span class="button special disabled">Special</span></li>
											<li><span class="button disabled">Default</span></li>
											<li><span class="button alt disabled">Alternate</span></li>
										</ul>
									</section>

									<section>
										<h4>Form</h4>
										<form method="post" action="#">
											<div class="row uniform">
												<div class="6u 12u(xsmall)">
													<input type="text" name="demo-name" id="demo-name" value="" placeholder="Name" />
												</div>
												<div class="6u 12u(xsmall)">
													<input type="email" name="demo-email" id="demo-email" value="" placeholder="Email" />
												</div>
											</div>
											<div class="row uniform">
												<div class="12u">
													<div class="select-wrapper">
														<select name="demo-category" id="demo-category">
															<option value="">- Category -</option>
															<option value="1">Manufacturing</option>
															<option value="1">Shipping</option>
															<option value="1">Administration</option>
															<option value="1">Human Resources</option>
														</select>
													</div>
												</div>
											</div>
											<div class="row uniform">
												<div class="4u 12u(medium)">
													<input type="radio" id="demo-priority-low" name="demo-priority" checked>
													<label for="demo-priority-low">Low Priority</label>
												</div>
												<div class="4u 12u(medium)">
													<input type="radio" id="demo-priority-normal" name="demo-priority">
													<label for="demo-priority-normal">Normal Priority</label>
												</div>
												<div class="4u 12u(medium)">
													<input type="radio" id="demo-priority-high" name="demo-priority">
													<label for="demo-priority-high">High Priority</label>
												</div>
											</div>
											<div class="row uniform">
												<div class="6u 12u(medium)">
													<input type="checkbox" id="demo-copy" name="demo-copy">
													<label for="demo-copy">Email me a copy of this message</label>
												</div>
												<div class="6u 12u(medium)">
													<input type="checkbox" id="demo-human" name="demo-human" checked>
													<label for="demo-human">I am a human and not a robot</label>
												</div>
											</div>
											<div class="row uniform">
												<div class="12u">
													<textarea name="demo-message" id="demo-message" placeholder="Enter your message" rows="6"></textarea>
												</div>
											</div>
											<div class="row uniform">
												<div class="12u">
													<ul class="actions">
														<li><input type="submit" value="Send Message" /></li>
														<li><input type="reset" value="Reset" class="alt" /></li>
													</ul>
												</div>
											</div>
										</form>
									</section>

									<section>
										<h4>Image</h4>
										<h5>Fit</h5>
										<span class="image fit"><img src="images/banner.jpg" alt="" /></span>
										<div class="box alt">
											<div class="row 50% uniform">
												<div class="4u"><span class="image fit"><img src="images/pic01.jpg" alt="" /></span></div>
												<div class="4u"><span class="image fit"><img src="images/pic02.jpg" alt="" /></span></div>
												<div class="4u"><span class="image fit"><img src="images/pic03.jpg" alt="" /></span></div>
											</div>
											<div class="row 50% uniform">
												<div class="4u"><span class="image fit"><img src="images/pic02.jpg" alt="" /></span></div>
												<div class="4u"><span class="image fit"><img src="images/pic03.jpg" alt="" /></span></div>
												<div class="4u"><span class="image fit"><img src="images/pic01.jpg" alt="" /></span></div>
											</div>
											<div class="row 50% uniform">
												<div class="4u"><span class="image fit"><img src="images/pic03.jpg" alt="" /></span></div>
												<div class="4u"><span class="image fit"><img src="images/pic01.jpg" alt="" /></span></div>
												<div class="4u"><span class="image fit"><img src="images/pic02.jpg" alt="" /></span></div>
											</div>
										</div>
										<h5>Left &amp; Right</h5>
										<p><span class="image left"><img src="images/avatar.jpg" alt="" /></span>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent.</p>
										<p><span class="image right"><img src="images/avatar.jpg" alt="" /></span>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent.</p>
									</section>

								</div>
							</section>
								-->

					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>&copy; Charleston Chiang 2017. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollzer.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>



		<!-- Start of StatCounter Code for Default Guide -->
			<script type="text/javascript">
			var sc_project=11576907; 
			var sc_invisible=1; 
			var sc_security="4cde62bc"; 
			var scJsHost = (("https:" == document.location.protocol) ?
			"https://secure." : "http://www.");
			document.write("<sc"+"ript type='text/javascript' src='" +
			scJsHost+
			"statcounter.com/counter/counter.js'></"+"script>");
			</script>
			<noscript><div class="statcounter"><a title="Web Analytics
			  Made Easy - StatCounter" href="http://statcounter.com/"
			  target="_blank"><img class="statcounter"
			  src="//c.statcounter.com/11576907/0/4cde62bc/1/" alt="Web
			  Analytics Made Easy - StatCounter"></a></div></noscript>
		<!-- End of StatCounter Code for Default Guide -->
	</body>
</html>